<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Standards &#8211; ISA agents</title>
	<atom:link href="category/standards/feed/" rel="self" type="application/rss+xml" />
	<link>http://isa-agents.org</link>
	<description>Standardizing metadata for scientific experiments</description>
	<lastBuildDate>Thu, 19 Apr 2018 15:23:29 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.5</generator>
<site xmlns="com-wordpress:feed-additions:1">113852667</site>	<item>
		<title>New ISA Model and Serialization Specifications released!</title>
		<link>http://isa-agents.org/2016/10/release-of-the-isa-specs/</link>
		<pubDate>Fri, 21 Oct 2016 11:28:57 +0000</pubDate>
		<dc:creator><![CDATA[David Johnson]]></dc:creator>
				<category><![CDATA[ISA]]></category>
		<category><![CDATA[Standards]]></category>
		<category><![CDATA[Isa]]></category>
		<category><![CDATA[specifications]]></category>

		<guid isPermaLink="false">http://isa-agents.org/?p=1221</guid>
		<description><![CDATA[Over the last few months, the ISA Team has been working hard on editing a long-awaited release of the ISA Model and Serialization Specifications 1.0. The original ISA-Tab specification was published as a Release Candidate document in 2008, documenting the initial work that forms the ISA framework, with a further update in 2009. Since then, we [&#8230;]]]></description>
				<content:encoded><![CDATA[<p>Over the last few months, the ISA Team has been working hard on editing a long-awaited release of the <a href="http://doi.org/10.5281/zenodo.163640">ISA Model and Serialization Specifications 1.0</a>.</p>
<p>The original ISA-Tab specification was published as a <a href="http://doi.org/10.5281/zenodo.161350">Release Candidate document</a> in 2008, documenting the initial work that forms the ISA framework, with a <a href="http://doi.org/10.5281/zenodo.161355">further update</a> in 2009. Since then, we have done work on a new serialization in JSON, <strong>ISA-JSON</strong>, and abstracted out the data model from both the tabular and JSON formats.</p>
<p><img data-attachment-id="1223" data-permalink="2016/10/release-of-the-isa-specs/isa-model-specs/" data-orig-file="https://isa-agents.org/wp-content/uploads/2016/10/isa-model-specs.png?fit=798%2C638" data-orig-size="798,638" data-comments-opened="0" data-image-meta="{&quot;aperture&quot;:&quot;0&quot;,&quot;credit&quot;:&quot;&quot;,&quot;camera&quot;:&quot;&quot;,&quot;caption&quot;:&quot;&quot;,&quot;created_timestamp&quot;:&quot;0&quot;,&quot;copyright&quot;:&quot;&quot;,&quot;focal_length&quot;:&quot;0&quot;,&quot;iso&quot;:&quot;0&quot;,&quot;shutter_speed&quot;:&quot;0&quot;,&quot;title&quot;:&quot;&quot;,&quot;orientation&quot;:&quot;0&quot;}" data-image-title="ISA Model and Specs diagram" data-image-description="" data-medium-file="https://isa-agents.org/wp-content/uploads/2016/10/isa-model-specs.png?fit=300%2C240" data-large-file="https://isa-agents.org/wp-content/uploads/2016/10/isa-model-specs.png?fit=625%2C500" class="alignleft size-full wp-image-1223" src="https://isa-agents.org/wp-content/uploads/2016/10/isa-model-specs.png?resize=625%2C500" alt="ISA Model and Specs diagram" width="625" height="500" srcset="https://isa-agents.org/wp-content/uploads/2016/10/isa-model-specs.png?w=798 798w, https://isa-agents.org/wp-content/uploads/2016/10/isa-model-specs.png?resize=300%2C240 300w, https://isa-agents.org/wp-content/uploads/2016/10/isa-model-specs.png?resize=768%2C614 768w, https://isa-agents.org/wp-content/uploads/2016/10/isa-model-specs.png?resize=624%2C499 624w" sizes="(max-width: 625px) 100vw, 625px" data-recalc-dims="1" /></p>
<p>The ISA Model and Serialization Specifications consist of three specification documents:</p>
<ol>
<li>ISA Abstract Model &#8211; a data model of ISA objects/entities and their relation to one another</li>
<li>ISA-Tab format &#8211; the tabular serialization format of the Abstract Model</li>
<li>ISA-JSON format &#8211; the JSON serialization format of the Abstract Model.</li>
</ol>
<p>The specifications are licensed under <a href="https://creativecommons.org/licenses/by-sa/4.0/">CC BY-SA 4.0</a> , and you can cite the specifications with:</p>
<p><em>Sansone, Susanna-Assunta, Rocca-Serra, Philippe, Gonzalez-Beltran, Alejandra, Johnson David &amp; the ISA Community. (2016, October 28). ISA Model and Serialization Specifications 1.0. Zenodo. <a href="http://doi.org/10.5281/zenodo.163640">http://doi.org/10.5281/zenodo.163640</a>.</em></p>
<p>To view the latest version online, please visit <a href="http://isa-specs.readthedocs.io/en/latest/">http://isa-specs.readthedocs.io/en/latest/</a></p>
]]></content:encoded>
		<post-id xmlns="com-wordpress:feed-additions:1">1221</post-id>	</item>
		<item>
		<title>Plant Science takes a focus on ISA</title>
		<link>http://isa-agents.org/2016/07/plant-science-takes-a-focus-on-isa/</link>
		<pubDate>Wed, 06 Jul 2016 10:49:36 +0000</pubDate>
		<dc:creator><![CDATA[David Johnson]]></dc:creator>
				<category><![CDATA[General]]></category>
		<category><![CDATA[ISA]]></category>
		<category><![CDATA[Standards]]></category>
		<category><![CDATA[workshops]]></category>
		<category><![CDATA[arabidopsis]]></category>
		<category><![CDATA[BioSharing]]></category>
		<category><![CDATA[FAIRdata]]></category>
		<category><![CDATA[Isa]]></category>
		<category><![CDATA[ISA infrastructure]]></category>
		<category><![CDATA[plant science]]></category>

		<guid isPermaLink="false">http://isa-agents.org/?p=1150</guid>
		<description><![CDATA[Back in April this year, Dr David Johnson from the ISA team gave a presentation on &#8220;Data Infrastructures to Foster Data Reuse&#8221; at a workshop on Integrating Large Data into Plant Science: From Big Data to Discovery hosted by GARnet (the UK network for Arabidopsis researchers) and Egenis (the Exeter Centre for the Study of [&#8230;]]]></description>
				<content:encoded><![CDATA[<p>Back in April this year, Dr David Johnson from the ISA team gave a presentation on &#8220;Data Infrastructures to Foster Data Reuse&#8221; at a workshop on <a href="http://www.plantsci.org.uk/events/integrating-large-data-plant-science-big-data-discovery"><em>Integrating Large Data into Plant Science: From Big Data to Discovery</em></a> hosted by <a href="http://www.garnetcommunity.org.uk/">GARnet</a> (the UK network for Arabidopsis researchers) and <a href="http://socialsciences.exeter.ac.uk/sociology/research/sts/egenis/">Egenis</a> (the Exeter Centre for the Study of the Life Sciences). The workshop was held at Dartington Hall in Devon, South West England, and was well attended by researchers from the plant and biological science community worldwide as well as representatives from industry from organisations such as <a href="https://www.syngenta.co.uk/">Syngenta</a>.</p>
<p>David presented on ISA, as well as on <a href="http://biosharing.org">biosharing.org</a>, as candidate data infrastructure resources for enabling data reuse in the plant sciences, as well as presenting an example of how one might encode high-throughput plant phenotyping in ISA tab.</p>
<p>We have observed the uptake of the ISA tab format across the broad range of life sciences, but view its adoption, with a view to making data <a href="http://www.nature.com/articles/sdata201618">FAIR</a> (Findable, Accessible, Interoperable and Reusable), in the plant sciences as essential for the field. In particular centres such as the UK’s <a href="http://www.plant-phenomics.ac.uk">National Plant Phenomics Centre</a> in Aberystwyth, Wales, could benefit hugely from adopting ISA where there are emerging challenges in data management, in particular as automation of data collection is a significant driver in modern plant-based research and agritech.</p>
<p>There are also existing data analysis platforms such as <a href="https://www.araport.org/">Araport</a> (the Arabidopsis information Portal), <a href="http://www.phoenixbioinformatics.org/tair/">TAIR</a> (The Arabidopsis Information Resources) and <a href="https://www.biodare.ed.ac.uk/">BioDare</a> (Biological Data Repository) that could benefit from standardizing their experimental data, as well as ongoing efforts to create open data resources in the plant sciences, such as the <a href="http://copo-project.org/">Collaborative Open Plant Omics</a> (COPO) project, that will be using the new ISA JSON format as native data objects.</p>
<p>You can check out David’s presentation <a href="http://www.slideshare.net/DavidJohnson478/garnet-workshop-on-integrating-large-data-into-plant-science">on SlideShare</a>.</p>
<p><iframe src="https://www.slideshare.net/slideshow/embed_code/key/EwVUZUiuUcN7HB" width="427" height="356" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; margin-bottom:5px; max-width: 100%;" allowfullscreen> </iframe> </p>
<div style="margin-bottom:5px"> <strong> <a href="https://www.slideshare.net/DavidJohnson478/garnet-workshop-on-integrating-large-data-into-plant-science" title="GARNet workshop on Integrating Large Data into Plant Science" target="_blank">GARNet workshop on Integrating Large Data into Plant Science</a> </strong> from <strong><a target="_blank" href="http://www.slideshare.net/DavidJohnson478">David Johnson</a></strong> </div>
]]></content:encoded>
		<post-id xmlns="com-wordpress:feed-additions:1">1150</post-id>	</item>
		<item>
		<title>Investigation/Study/Assay, Hacks/Coffee/Cakes</title>
		<link>http://isa-agents.org/2015/07/investigation-study-assay-hacks-coffee-cakes/</link>
		<pubDate>Sun, 26 Jul 2015 14:18:22 +0000</pubDate>
		<dc:creator><![CDATA[David Johnson]]></dc:creator>
				<category><![CDATA[Improvements]]></category>
		<category><![CDATA[ISA]]></category>
		<category><![CDATA[Standards]]></category>
		<category><![CDATA[workshops]]></category>

		<guid isPermaLink="false">http://isa-agents.org/?p=988</guid>
		<description><![CDATA[In this post, Dr David Johnson gives his reflection on an ISA specification hackathon held in July 2015, in advance of joining the ISA team at Oxford as a research software engineer. Last week I joined my prospective colleagues at the Oxford University e-Research Centre (OeRC) with some of their collaborators to thresh out an evolution [&#8230;]]]></description>
				<content:encoded><![CDATA[<p><strong><em>In this post, Dr David Johnson gives his reflection on an ISA specification hackathon held in July 2015, in advance of joining the ISA team at Oxford as a research software engineer.</em></strong></p>
<p>Last week I joined my prospective colleagues at the <a href="http://www.oerc.ox.ac.uk/" target="_blank">Oxford University e-Research Centre</a> (OeRC) with some of their collaborators to thresh out an evolution of the ISA (Investigation/Study/Assay) metadata tracking framework. I will be joining the ISA development team at Oxford from September this year, which is a new phase in my career that I am very much looking forward to.</p>
<p>ISA consists of a model specification that describes its key concept elements and structure, while implementations of the specification are also developed by the ISA team. The framework aims to facilitate standards-compliant collection, curation, management and reuse of datasets in the life sciences. The first version of the specification, a <a href="http://isatab.sourceforge.net/format.html" target="_blank">Release Candidate from 2008</a>, is implemented as the ISA-Tab (tabular) format &#8211; a table-based format that many working in the life sciences are used to, where data is abundantly stored and manipulated in spreadsheets. More recently ISA can also be converted to RDF via <a href="http://isa-agents.github.io/linkedISA/" target="_blank">linkedISA</a>.</p>
<blockquote class="twitter-tweet" data-width="550">
<p lang="en" dir="ltr">Serious ISA 2.0 core vs. extension spec hacking. Powered by cake. <a href="https://twitter.com/hashtag/isamodel?src=hash">#isamodel</a> <a href="https://twitter.com/hashtag/isa_hack?src=hash">#isa_hack</a> <a href="https://twitter.com/isaagents">@isaagents</a> <a href="http://t.co/W9c6ASilOA">pic.twitter.com/W9c6ASilOA</a></p>
<p>&mdash; Robert Davey (@froggleston) <a href="https://twitter.com/froggleston/status/623488074534797312">July 21, 2015</a></p></blockquote>
<p><script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script></p>
<p>While I have yet to officially join the ISA team (I am currently on a short sabbatical since leaving a research post at Imperial College London) I was invited to attend a 3-day workshop in Oxford to review and make new amendments to the ISA specification <strong>towards a version 2.0 release</strong>. The workshop, an <a href="http://iechor-uk.org/" target="_blank">IECHOR UK</a> event, was billed as the <i><a href="http://iechor-uk.org/node-events/201cisa-as-a-fair-research-object201d-hack-the-spec-event-1" target="_blank">“ISA as a FAIR research object” Hack-the-Spec event</a></i>. We were joined by representatives from <a href="http://www.tgac.ac.uk/" target="_blank">The Genome Analysis Centre</a>, the <a href="http://www.ebi.ac.uk/" target="_blank">European Bioinformatics Institute</a>, <a href="http://www.leidenuniv.nl/" target="_blank">Leiden</a>, <a href="http://www.manchester.ac.uk/" target="_blank">Manchester</a> and <a href="http://www.birmingham.ac.uk/" target="_blank">Birmingham</a> universities and even a group visiting from my home-town of Hong Kong, from the <a href="http://www.gigasciencejournal.com/" target="_blank">GigaScience journal</a> that was launched by <a href="http://www.genomics.cn/" target="_blank">Beijing Genomics Institute</a> in 2012. We also were joined online by a number of researchers dialling in via Google hangouts from various sites in Europe.</p>
<p>As a workshop report will come out in due course I won&#8217;t get into the detail of the outcomes, but broadly the discussions focused around:</p>
<ul>
<li>Evolving ISA to enable <a href="http://www.researchobject.org/" target="_blank">FAIR</a> (Findable, Accessible, Interoperable, Reusable) research objects</li>
<li>Fixing ambiguities, missing structures and elements in ISA 1.0</li>
<li>Enabling integration of standard identification schemes such as <a href="http://orcid.org/" target="_blank">ORCID</a></li>
<li>Redefining the spec to define the &#8216;core&#8217; ISA elements and separating out domain specific &#8216;extensions&#8217;</li>
<li>Specifying conventions, mechanisms, and best practices for developing extensions to this new &#8216;ISA core&#8217;.</li>
</ul>
<p>What was clear was that there was plenty of scope for evolving ISA from various parts of the user community. By abstracting out the core ISA specification, what we need now is contributions from a diverse range of exemplar projects to ensure that the core is truly interoperable. To this end, we are now encouraging communities to share their ISA templates along with exemplar experiments and start building a repository of extensions in the <a href="http://www.isa-commons.org/" target="_blank">ISA commons</a> website. In the meantime the ISA team will be formalising the ISA core and developing new reference implementations in tabular and JSON formats and supporting agents. We hope to have a draft specification presented to the community in the fall of 2015.</p>
<p>Apart from the 3 days of discussions fuelled by much coffee and cake, we did also find some time in the evenings to get out to enjoy the sunshine and enjoy a couple of Oxford&#8217;s wonderful restaurants&#8230;</p>
<blockquote class="twitter-tweet" data-width="550">
<p lang="en" dir="ltr">A well deserved hackathon dinner at The Folly <a href="https://twitter.com/hashtag/Oxford?src=hash">#Oxford</a> <a href="https://twitter.com/hashtag/iEchorNodeUK?src=hash">#iEchorNodeUK</a> <a href="https://twitter.com/hashtag/isa_hack?src=hash">#isa_hack</a> <a href="http://t.co/9zUcoqWwE2">pic.twitter.com/9zUcoqWwE2</a></p>
<p>&mdash; Dr David Johnson (@NuDataScientist) <a href="https://twitter.com/NuDataScientist/status/623572997778096128">July 21, 2015</a></p></blockquote>
<p><script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script></p>
<p>One of my key takeaways from the workshop, apart from having a crash course into the ISA spec that I will be working on in the coming months, is the importance of going through the community engagement process when developing a data specification. As with engineering software, we need to make sure we are building the right thing. Soliciting feedback is not a vanity exercise or even a political exercise, but an essential part of a carefully-managed process to ensure we evolve the specification to fulfil the changing needs of the people that matter &#8211; the user community.</p>
<p>Find out more about ISA here at <a href="http://isa-agents.org/">www.isa-agents.org</a> and about the wider community&#8217;s efforts at <a href="http://isacommons.org/">isacommons.org</a></p>
<p>Catch up tweets about this ISA 2.0 specification workshop by searching the hashtags <a href="https://twitter.com/hashtag/isa_hack" target="_blank">#isa_hack</a> and<a href="https://twitter.com/hashtag/iEchorNodeUK" target="_blank">#iEchorNodeUK</a></p>
]]></content:encoded>
		<post-id xmlns="com-wordpress:feed-additions:1">988</post-id>	</item>
		<item>
		<title>Introducing OntoMaton  &#8211; Ontology Search &#038; Tagging for Google Spreadsheets</title>
		<link>http://isa-agents.org/2012/07/introducing-ontomaton-ontology-search-tagging-for-google-spreadsheets/</link>
		<comments>http://isa-agents.org/2012/07/introducing-ontomaton-ontology-search-tagging-for-google-spreadsheets/#comments</comments>
		<pubDate>Fri, 13 Jul 2012 17:50:53 +0000</pubDate>
		<dc:creator><![CDATA[isaagents]]></dc:creator>
				<category><![CDATA[ISA]]></category>
		<category><![CDATA[ISAcreator]]></category>
		<category><![CDATA[Ontologies]]></category>
		<category><![CDATA[Standards]]></category>
		<category><![CDATA[Web]]></category>
		<category><![CDATA[Google docs]]></category>
		<category><![CDATA[Google Spreadsheets]]></category>
		<category><![CDATA[ISA-Ta]]></category>
		<category><![CDATA[ISAagents]]></category>
		<category><![CDATA[NCBO Bioportal]]></category>
		<category><![CDATA[OntoMaton]]></category>

		<guid isPermaLink="false">http://isaagents.wordpress.com/?p=366</guid>
		<description><![CDATA[We are happy to announce the release of OntoMaton, a agent which allows users to search for ontology terms and tag free text right in Google Spreadsheets. This post will serve to introduce you to the agent, how it works and how it can make it easier for users to use ontologies in a pervasive, [&#8230;]]]></description>
				<content:encoded><![CDATA[<p><a href="https://isa-agents.org/wp-content/uploads/2012/07/ontomaton.png"><img class="aligncenter size-medium wp-image-367" title="ontomaton" src="https://isa-agents.org/wp-content/uploads/2012/07/ontomaton.png?resize=300%2C80" alt="" width="300" height="80" data-recalc-dims="1" /></a></p>
<p>We are happy to announce the release of OntoMaton, a agent which allows users to <strong>search for ontology terms</strong> and <strong>tag free text right in Google Spreadsheets</strong>. This post will serve to introduce you to the agent, how it works and how it can make it easier for users to use ontologies in a pervasive, powerful and collaborative environment, complementing existing work from our team in the creation of ISAcreator.</p>
<p><strong>How it looks</strong></p>
<p>OntoMaton is available from the Google Script Gallery and when installed provides a menu as shown below.</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2012/07/menu.png"><img class="aligncenter size-full wp-image-364" title="menu" src="https://isa-agents.org/wp-content/uploads/2012/07/menu.png?resize=625%2C275" alt="" width="625" height="275" data-recalc-dims="1" /></a></p>
<p>From the menu you may access two resources part of OntoMaton: ontology search and ontology tagging. There is also an &#8216;about&#8217; option.</p>
<p><strong>Ontology Search</strong></p>
<p><a href="https://isa-agents.org/wp-content/uploads/2012/07/ontomaton-search-shot.png"><img class="aligncenter size-full wp-image-361" title="ontomaton-search-shot" src="https://isa-agents.org/wp-content/uploads/2012/07/ontomaton-search-shot.png?resize=625%2C681" alt="" width="625" height="681" data-recalc-dims="1" /></a></p>
<p><strong>Ontology Tagging</strong></p>
<p><a href="https://isa-agents.org/wp-content/uploads/2012/07/ontomaton-tag-shot.png"><img class="aligncenter size-full wp-image-360" title="ontomaton-tag-shot" src="https://isa-agents.org/wp-content/uploads/2012/07/ontomaton-tag-shot.png?resize=625%2C476" alt="" width="625" height="476" data-recalc-dims="1" /></a></p>
<p><strong>Behind the scenes: <em>restricting the ontology search space</em></strong></p>
<p>If a sheet named &#8220;restrictions&#8221; is in your spreadsheet, OntoMaton will consult it to determine if the currently selected column/row name has a narrowed ontology search space. This makes it quicker to search BioPortal, allows for restriction of the user&#8217;s result space to make easier the process of selecting a term.</p>
<p><strong>Behind the scenes: <em>extra information about the terms you select</em></strong></p>
<p>For every term you select, it&#8217;s full details are recorded in a &#8220;terms&#8221; sheet. This makes it possible to use OntoMaton in any spreadsheet and all provenance information (including URIs, ontology source and version) for selected ontology terms will be immediately available for use when exposing your records to the <strong>linked data world!</strong></p>
<p><strong>Installing</strong></p>
<p>To install, create a new google spreadsheet, then go to the menu <strong>agents</strong> &gt; <strong>script gallery</strong>. In the script gallery, search for <em>ontology</em> or <em>ontomaton </em>and you&#8217;ll get the following result pane.</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2012/07/scriptgallery.png"><img class="aligncenter size-full wp-image-372" title="scriptgallery" src="https://isa-agents.org/wp-content/uploads/2012/07/scriptgallery.png?resize=625%2C480" alt="" width="625" height="480" data-recalc-dims="1" /></a></p>
<p>Click on &#8216;install&#8217; and this will install the scripts inside your spreadsheet. Then there is one more and final step to follow for installation. You have to click again on <strong>agents</strong> &gt; <strong>script manager </strong>and you&#8217;ll be presented with something like that shown in the image below.</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2012/07/scriptmanager.png"><img class="aligncenter size-full wp-image-371" title="scriptmanager" src="https://isa-agents.org/wp-content/uploads/2012/07/scriptmanager.png?resize=625%2C451" alt="" width="625" height="451" data-recalc-dims="1" /></a></p>
<p>OntoMaton contains lots of functions, but the only one you need to worry about in order to run the program is the <strong><em>onOpen</em></strong> function. Click this then click on <em>run</em> and the OntoMaton menu will be installed in your menu bar. From here you&#8217;ll be able to access the <strong>ontology search</strong> and <strong>ontology tagging</strong> functions.</p>
<p>Let us know what you think! New releases will come soon to fix any problems you may identify, please submit all &#8216;bugs&#8217; and feature requests through <a href="https://github.com/ISA-agents/OntoMaton/issues"><strong>https://github.com/ISA-agents/OntoMaton/issues</strong></a></p>
<p>OntoMaton inherently supports ISA-Tab files too. So if you have an investigation file it will automatically add ontology sources to the ONTOLOGY SOURCE REFERENCE block. Also, if you have Term Source Ref and Term Source Accession after a column, OntoMaton will automatically populate these columns for you.</p>
<p>Also, the following table provides a quick review of available agents attempting to mix spreadsheets and access to vocabulary servers:</p>
<table>
<tbody>
<tr>
<td></td>
<td>
<p dir="ltr">domain</p>
</td>
<td>
<p dir="ltr">automated</p>
<p dir="ltr">annotation</p>
</td>
<td>
<p dir="ltr">ontology search/lookup</p>
</td>
<td>
<p dir="ltr">versioning*</p>
</td>
<td>
<p dir="ltr">collaboration</p>
</td>
</tr>
<tr>
<td>
<p dir="ltr">RightField</p>
</td>
<td>
<p dir="ltr">general</p>
</td>
<td>
<p dir="ltr"><span style="color:red;">✘</span></p>
</td>
<td>
<p dir="ltr"><span style="color:green;">✓</span></p>
</td>
<td>
<p dir="ltr"><span style="color:red;">✘</span></p>
</td>
<td>
<p dir="ltr"><span style="color:red;">✘</span></p>
</td>
</tr>
<tr>
<td>
<p dir="ltr">ISA creator</p>
</td>
<td>
<p dir="ltr">multiomics</p>
</td>
<td>
<p dir="ltr"><span style="color:green;">✓</span></p>
</td>
<td>
<p dir="ltr"><span style="color:green;">✓</span></p>
</td>
<td>
<p dir="ltr"><span style="color:red;">✘</span></p>
</td>
<td>
<p dir="ltr"><span style="color:red;">✘</span></p>
</td>
</tr>
<tr>
<td>
<p dir="ltr">Proteome Harvest PRIDE</p>
</td>
<td>
<p dir="ltr">proteomics</p>
</td>
<td>
<p dir="ltr"><span style="color:red;">✘</span></p>
</td>
<td>
<p dir="ltr"><span style="color:green;">✓</span></p>
</td>
<td>
<p dir="ltr"><span style="color:red;">✘</span></p>
</td>
<td>
<p dir="ltr"><span style="color:red;">✘</span></p>
</td>
</tr>
<tr>
<td>
<p dir="ltr">Annotare</p>
</td>
<td>
<p dir="ltr">transcriptomics</p>
</td>
<td>
<p dir="ltr"><span style="color:red;">✘</span></p>
</td>
<td>
<p dir="ltr"><span style="color:red;">✘</span></p>
</td>
<td>
<p dir="ltr"><span style="color:green;">✓</span></p>
</td>
<td>
<p dir="ltr"><span style="color:red;">✘</span></p>
</td>
</tr>
<tr>
<td>
<p dir="ltr">OntoMaton</p>
</td>
<td>
<p dir="ltr">general</p>
</td>
<td>
<p dir="ltr"><span style="color:green;">✓</span></p>
</td>
<td>
<p dir="ltr"><span style="color:green;">✓</span></p>
</td>
<td>
<p dir="ltr"><span style="color:green;">✓</span></p>
</td>
<td>
<p dir="ltr"><span style="color:green;">✓</span></p>
</td>
</tr>
</tbody>
</table>
<ul>
<li>by versioning we refer to managing of user edits throughout the annotation process.</li>
</ul>
<p>We hope you enjoy this new feature!</p>
<p>The ISA team</p>
<p><i>Addendum:</i></p>
<p><span style="color:red;">Safari 6 users</span>, be aware you will have to activate the &#8216;developer menu&#8217; from the Advanced Item in the Safari &#8216;Preferences&#8217; menu item. Once activated, go to menu &#8216;Develop&#8217; and navigate to &#8216;User Agent&#8217; item and select &#8216;Safari 5.1.7&#8217; for enabling the browser to work with Google Spreadsheet. (Thanks to <a href="https://github.com/rpyzh">rpyzh</a> for reporting the issue, see <a href="https://github.com/ISA-agents/OntoMaton/issues/1">here</a>)</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2012/08/ontomaton-safari-6-preference.png"><img class="aligncenter size-full wp-image-372" title="Ontomaton-Safari-6-preference" src="https://isa-agents.org/wp-content/uploads/2012/08/ontomaton-safari-6-preference.png?resize=625%2C480" alt="" width="625" height="480" data-recalc-dims="1" /></a></p>
<p><a href="https://isa-agents.org/wp-content/uploads/2012/08/ontomaton-safari-6-developer-menu-display.png"><img class="aligncenter" title="Ontomaton-Safari-6-developer-menu-display" src="https://isa-agents.org/wp-content/uploads/2012/08/ontomaton-safari-6-developer-menu-display.png?resize=450%2C300" alt="" width="450" height="300" data-recalc-dims="1" /></a></p>
<p><a href="https://isa-agents.org/wp-content/uploads/2012/08/ontomaton-safari-6-user-agent-selection.png"><img class="aligncenter size-full wp-image-372" title="ontomaton-safari-6-user-agent-selection" src="https://isa-agents.org/wp-content/uploads/2012/08/ontomaton-safari-6-user-agent-selection.png?resize=63%2C48" alt="" width="63" height="48" data-recalc-dims="1" /></a></p>
]]></content:encoded>
			<wfw:commentRss>http://isa-agents.org/2012/07/introducing-ontomaton-ontology-search-tagging-for-google-spreadsheets/feed/</wfw:commentRss>
		<slash:comments>14</slash:comments>
	<post-id xmlns="com-wordpress:feed-additions:1">366</post-id>	</item>
		<item>
		<title>Toward interoperable bioscience data published in Nature Genetics</title>
		<link>http://isa-agents.org/2012/01/toward-interoperable-bioscience-data-published-in-nature-genetics/</link>
		<comments>http://isa-agents.org/2012/01/toward-interoperable-bioscience-data-published-in-nature-genetics/#comments</comments>
		<pubDate>Fri, 27 Jan 2012 20:02:56 +0000</pubDate>
		<dc:creator><![CDATA[isaagents]]></dc:creator>
				<category><![CDATA[ISA]]></category>
		<category><![CDATA[ISAcommons]]></category>
		<category><![CDATA[Publication]]></category>
		<category><![CDATA[Standards]]></category>

		<guid isPermaLink="false">http://isaagents.wordpress.com/?p=311</guid>
		<description><![CDATA[Our commentary, named &#8220;Toward interoperable bioscience data&#8221; has just been released in Nature Genetics. In it, we focus on working towards an ISA commons, where one day we&#8217;ll all be able to share our experimental metadata in one common, easy to use format, supporting minimal information checklists and ontologies (via the ISA-Tab format and respective [&#8230;]]]></description>
				<content:encoded><![CDATA[<p><a href="http://www.nature.com/ng/journal/v44/n2/full/ng.1054.html"><img class="aligncenter" title="Nature Genetics cover, Feb 2012" src="https://i0.wp.com/www.nature.com/ng/journal/v44/n2/images/homecover.gif?resize=131%2C170" alt="" width="131" height="170" data-recalc-dims="1" /></a></p>
<p>Our commentary, named &#8220;<a href="http://www.nature.com/ng/journal/v44/n2/full/ng.1054.html" target="_blank">Toward interoperable bioscience data</a>&#8221; has just been released in Nature Genetics. In it, we focus on working towards an <a href="http://isacommons.org" target="_blank">ISA commons</a>, where one day we&#8217;ll all be able to share our experimental metadata in one common, easy to use format, supporting minimal information checklists and ontologies (via the <a href="http://isa-agents.org" target="_blank">ISA-Tab format and respective agents</a>).</p>
<p><a href="http://www.nature.com/ng/journal/v44/n2/full/ng.1099.html" target="_blank">The editor speaks about the paper</a>, paraphrased here:</p>
<blockquote><p>&#8220;Reformatting data is a full-time job for many researchers, even before the minimum reporting guidelines, terminologies and formats of each field are taken into consideration. In this issue, we present a Commentary and a Perspective suggesting solutions to these problems that have been developed by a process of community consultation and open review to which the journal was a party. In the Commentary, Susanna-Assunta Sansone and colleagues identify one central problem, namely that “most repositories are designed for specific assay types, necessitating the fragmentation of complex datasets,” and they offer a unified view of the metadata formatting that will be needed to ensure that biomedical research datasets become interoperable. This solution is the overarching ISA framework, where the acronym stands for &#8216;Investigation&#8217; (the project context), &#8216;Study&#8217; (a unit of research) and &#8216;Assay&#8217; (analytical measurement) (<a href="http://www.nature.com/ng/journal/v44/n2/full/ng.1054.html">p 121</a>). This proposal shifts the sets of reporting standards agreed upon by each community into the infrastructure and formatting of the data files themselves. Sansone and colleagues also list a set of participant communities that can pioneer the approach and teach by example.&#8221;</p></blockquote>
<p>Many thanks to all who contributed to the paper and to the growing success of ISA (commons and agents).</p>
]]></content:encoded>
			<wfw:commentRss>http://isa-agents.org/2012/01/toward-interoperable-bioscience-data-published-in-nature-genetics/feed/</wfw:commentRss>
		<slash:comments>1</slash:comments>
	<post-id xmlns="com-wordpress:feed-additions:1">311</post-id>	</item>
		<item>
		<title>Browse &#038; search OLS and BioPortal Seamlessly with ISAcreator and ISAcreator configurator</title>
		<link>http://isa-agents.org/2010/06/browse-search-ols-and-bioportal-seamlessly/</link>
		<pubDate>Tue, 15 Jun 2010 16:29:31 +0000</pubDate>
		<dc:creator><![CDATA[isaagents]]></dc:creator>
				<category><![CDATA[ISAcreator]]></category>
		<category><![CDATA[ISAcreator configurator]]></category>
		<category><![CDATA[Ontologies]]></category>
		<category><![CDATA[Standards]]></category>

		<guid isPermaLink="false">http://isaagents.wordpress.com/?p=27</guid>
		<description><![CDATA[For browsing and querying OLS and BioPortal seamlessly, ISAcreator and it&#8217;s associated configurator have done much to make life easier for it&#8217;s users. ISAcreator configurator and it&#8217;s use for ontologies ISAcreator Configurator is a agent used by community experts or curators to detail the Minimum Information (via MIBBI for instance) that users should enter to [&#8230;]]]></description>
				<content:encoded><![CDATA[<p>For browsing and querying <strong><a title="OLS" href="http://www.ebi.ac.uk/ontology-lookup/" target="_blank">OLS</a></strong> and <strong><a title="BioPortal" href="http://bioportal.bioontology.org/">BioPortal</a></strong> seamlessly, <strong>ISAcreator</strong> and it&#8217;s associated <strong>configurator</strong> have done much to make life easier for it&#8217;s users.</p>
<p><strong><em>ISAcreator configurator</em> and it&#8217;s use for ontologies</strong></p>
<p>ISAcreator Configurator is a agent used by community experts or curators to detail the Minimum Information (via MIBBI for instance) that users should enter to describe their experiment. As part of this, curators can define which fields (e.g. Sample Name) are required, whether or not they require ontology terms and if so, which ontologies (and parts of the ontologies) users should be pointed to.</p>
<div id="attachment_31" style="width: 410px" class="wp-caption aligncenter"><a href="https://isa-agents.org/wp-content/uploads/2010/06/branches_recorded_for_searching_ontologies.png"><img class="size-medium wp-image-31" title="ISAcreator Configurator" src="https://isa-agents.org/wp-content/uploads/2010/06/branches_recorded_for_searching_ontologies.png?resize=400%2C266" alt="" width="400" height="266 .667" data-recalc-dims="1" /></a><p class="wp-caption-text">ISAcreator Configurator &#8211; allows curators to define &#8216;checklists&#8217;: these are the fields required to describe an experiment from start to finish.</p></div>
<p>The ISAcreator configurator provides both the ability to <strong>browse ontologie</strong>s and to <strong>search</strong> them, so as to allow <strong>curators</strong> to select which ontologies and which parts of these ontologies should be suggested to users whenever they annotate a field in the <strong>ISAcreator</strong>.</p>
<p><strong>Browsing ontologies</strong></p>
<p>Users can browse ontologies in OLS and BioPortal via an easy to use GUI. The browsing is done on the fly by accessing hierarchy web services from both the OLS and BioPortal. An extensible code base means that other ontology resource (should they become available) can be added at any point.</p>
<div id="attachment_29" style="width: 410px" class="wp-caption aligncenter"><a href="https://isa-agents.org/wp-content/uploads/2010/06/browse_ontologies.png"><img class="size-medium wp-image-29" title="Browse ChEBI from OLS" src="https://isa-agents.org/wp-content/uploads/2010/06/browse_ontologies.png?resize=400%2C266" alt="" width="400" height="266 .667" data-recalc-dims="1" /></a><p class="wp-caption-text">Browse ChEBI from OLS</p></div>
<div id="attachment_28" style="width: 410px" class="wp-caption aligncenter"><a href="https://isa-agents.org/wp-content/uploads/2010/06/browse_bioportal_ontologies.png"><img class="size-medium wp-image-28" title="Browsing ontologies from BioPortal" src="https://isa-agents.org/wp-content/uploads/2010/06/browse_bioportal_ontologies.png?resize=400%2C266" alt="" width="400" height="266 .667" data-recalc-dims="1" /></a><p class="wp-caption-text">Browsing OBI (Ontology for Biomedical Investigations) from BioPortal</p></div>
<p><strong>Search within ontologies</strong></p>
<div id="attachment_30" style="width: 410px" class="wp-caption aligncenter"><a href="https://isa-agents.org/wp-content/uploads/2010/06/search_within_ontologies.png"><img class="size-medium wp-image-30" title="Search within ontologies" src="https://isa-agents.org/wp-content/uploads/2010/06/search_within_ontologies.png?resize=400%2C266" alt="" width="400" height="266 .667" data-recalc-dims="1" /></a><p class="wp-caption-text">Search within ontologies</p></div>
<p>Users can search within ontologies and then locate the ontology in the entire ontology hierarchy. This functionality is provided to aid curators in finding the appropriate branch of an ontology to restrict users to.</p>
<p><strong><em>ISAcreator</em> and how it makes ontology selection easier for users</strong></p>
<p>ISAcreator is a agent used by the community who often don&#8217;t know much about ontologies or the minimum information required to describe an experiment. The ISAcreator configurator produces XML describing the minimum information and the ontologies to use so that users need not worry about what to enter.</p>
<div id="attachment_34" style="width: 410px" class="wp-caption aligncenter"><a href="https://isa-agents.org/wp-content/uploads/2010/06/main_menu.png"><img class="size-medium wp-image-34" title="ISAcreator's main menu" src="https://isa-agents.org/wp-content/uploads/2010/06/main_menu.png?resize=400%2C276" alt="" width="400" height="276 .667" data-recalc-dims="1" /></a><p class="wp-caption-text">ISAcreator&#8217;s main menu</p></div>
<p>The user interface needs to be easy to use, and ontology selection should be seamless. The user needs not know which ontology resource they are browsing, but ontologies should be presented to them in a way which makes selection straightforward. ISAcreator achieves this by automatically prompting users for ontology terms whenever they are set to be required by the ISAcreator configurator.</p>
<div id="attachment_35" style="width: 410px" class="wp-caption aligncenter"><a href="https://isa-agents.org/wp-content/uploads/2010/06/ontology_lookup_3.png"><img class="size-medium wp-image-35" title="Ontology lookup" src="https://isa-agents.org/wp-content/uploads/2010/06/ontology_lookup_3.png?resize=400%2C276" alt="" width="400" height="276 .667" data-recalc-dims="1" /></a><p class="wp-caption-text">Users are automatically prompted for Ontology terms</p></div>
<div id="attachment_36" style="width: 410px" class="wp-caption aligncenter"><a href="https://isa-agents.org/wp-content/uploads/2010/06/ontology_lookup_2.png"><img class="size-medium wp-image-36" title="Ontology lookup" src="https://isa-agents.org/wp-content/uploads/2010/06/ontology_lookup_2.png?resize=400%2C276" alt="" width="400" height="276 .667" data-recalc-dims="1" /></a><p class="wp-caption-text">Users are presented with the full search result from both OLS and Bioportal in a standard form.</p></div>
<p>In summary, the functionalities inside both ISAcreator and it&#8217;s associated configurator make browsing and searching both OLS and BioPortal much simpler for users! An API with many advanced functionalities, building upon BioPortal and OLS services will be released soon as well as a standalone ontology browser and searcher!</p>
<p>Download ISAcreator and the ISAcreator configurator now from <a href="http://isatab.sf.net">the ISAtab sourceforge site</a>!</p>
]]></content:encoded>
		<post-id xmlns="com-wordpress:feed-additions:1">27</post-id>	</item>
	</channel>
</rss>
