<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>agbeltran &#8211; ISA agents</title>
	<atom:link href="author/agbeltran/feed/" rel="self" type="application/rss+xml" />
	<link>http://isa-agents.org</link>
	<description>Standardizing metadata for scientific experiments</description>
	<lastBuildDate>Thu, 19 Apr 2018 15:23:29 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.5</generator>
<site xmlns="com-wordpress:feed-additions:1">113852667</site>	<item>
		<title>Join the funFAIR!</title>
		<link>http://isa-agents.org/2016/03/join-the-funfair/</link>
		<pubDate>Tue, 15 Mar 2016 22:17:25 +0000</pubDate>
		<dc:creator><![CDATA[agbeltran]]></dc:creator>
				<category><![CDATA[ISA]]></category>
		<category><![CDATA[FAIRdata]]></category>

		<guid isPermaLink="false">http://isa-agents.org/?p=1056</guid>
		<description><![CDATA[Today, March 15 2016, the FAIR Guiding Principles for scientific data management and stewardship were formally published in the Nature Publishing Group journal Scientific Data. The problem the FAIR Principles address is the lack of widely shared, clearly articulated, and broadly applicable best practices around the publication of scientific data. While the history of scholarly [&#8230;]]]></description>
				<content:encoded><![CDATA[<p><b>Today, March 15 2016, the FAIR Guiding Principles for scientific data management and stewardship were formally published in the Nature Publishing Group journal <a href="http://www.nature.com/articles/sdata201618" target="new"><i>Scientific Data</i></a>. The problem the FAIR Principles address is the lack of widely shared, clearly articulated, and broadly applicable best practices around the publication of scientific data. While the history of scholarly publication in journals is long and well established, the same cannot be said of formal data publication. Yet, data could be considered the primary output of scientific research, and its publication and reuse is necessary to ensure validity, reproducibility, and to drive further discoveries. The FAIR Principles address these needs by providing a precise and measurable set of qualities a good data publication should exhibit – qualities that ensure that the data is Findable, Accessible, Interoperable, and Reusable (<a href="http://www.dtls.nl/fair-data/" target="new">FAIR</a>).</b></p>
<p>The <a href="http://isacommons.org">ISA infrastructure project</a> and <a href="http://biosharing.org">BioSharing</a> registry of standards, databases and policies are both part of this community in which we strive to make data FAIR. Do join us in these efforts!</p>
<p>For more information,<a href="https://t.co/DTMGtqTk8r"> read the paper</a> and see the <a href="http://www.dtls.nl/the-fair-principles-herald-more-open-transparent-and-reusable-scientific-data/">press release at the Dutch Tech Centre for Life Sciences</a>. <a href="https://t.co/DTMGtqTk8r" rel="attachment wp-att-1060"><img class="aligncenter wp-image-1060 size-full" src="https://isa-agents.org/wp-content/uploads/2016/03/CcDXuERWAAEPM4c.jpg?resize=594%2C746" alt="scidata fair" width="594" height="746" data-recalc-dims="1" /></a></p>
]]></content:encoded>
		<post-id xmlns="com-wordpress:feed-additions:1">1056</post-id>	</item>
		<item>
		<title>The ISA team is growing!</title>
		<link>http://isa-agents.org/2016/02/the-isa-team-is-growing/</link>
		<pubDate>Mon, 08 Feb 2016 22:16:00 +0000</pubDate>
		<dc:creator><![CDATA[agbeltran]]></dc:creator>
				<category><![CDATA[ISA]]></category>

		<guid isPermaLink="false">http://isa-agents.org/?p=998</guid>
		<description><![CDATA[We are very happy to announced that Dr David Johnson and Dr Massimiliano Izzo have joined the ISA team as a Research Software Engineers, last year and this year, respectively. David and Massi are both great additions to the team. A few words about their past experience&#8230; David completed his PhD at the University of [&#8230;]]]></description>
				<content:encoded><![CDATA[<p>We are very happy to announced that Dr David Johnson and Dr Massimiliano Izzo have joined the ISA team as a Research Software Engineers, last year and this year, respectively.</p>
<p>David and Massi are both great additions to the team. A few words about their past experience&#8230;</p>
<div style="width: 209px" class="wp-caption alignnone"><img class="" src="https://isa-agents.org/wp-content/uploads/2015/09/dj_profile.png?resize=199%2C200" alt="David" width="199" height="200" data-recalc-dims="1" /><p class="wp-caption-text">David</p></div>
<p>David completed his PhD at the University of Reading (UK) and before joining us at the University of Oxford e-Research Centre (OeRC), he worked at Imperial College London where he was a founding member of the Data Science Institute. Prior to that he worked in the Department of Computer Science at Oxford University, where he was part of an FP7 project developing interoperable cancer model databases, and also in the Evolutionary Biology Group at the University of Reading where he developed high-performance computing software for phylogenetics. He serves on the technical programme committees of a number of international conferences including the <i>International Conference on Computational Science</i> series and on the editorial board of the journal <i>Cancer Informatics</i>.</p>
<div style="width: 221px" class="wp-caption alignnone"><img class="" src="https://isa-agents.org/wp-content/uploads/2016/01/MI.jpg?resize=211%2C211" alt="Massi" width="211" height="211" data-recalc-dims="1" /><p class="wp-caption-text">Massi</p></div>
<p>Massi completed PhD studies in Biomedical Engineering at the University of Genoa (Italy). His main interests are in the design and development of innovative data models for Life Sciences, structured/unstructured data management and full-stack software development (JavaScript all the way!). Before joining the OeRC, he was a Research Collaborator at the Giannini Gaslini Institute, in Genoa (Italy) where he developed distributed data management systems for Integrated Biobanking Management, mostly targeted to Paediatric Tumours. In his free time, Massi enjoys reading (mostly speculative fiction novels), gazing at the ceiling while lying on the sofa, and wander aimlessly in bookshops and cafes.</p>
<p>You can follow all their code contributions to <a href="http://github.com/ISA-agents">ISA-agents</a> through their Github profiles: <a href="http://github.com/djcomlab">djcomlab</a> and <a href="http://github.com/zigur">zigur</a>.</p>
]]></content:encoded>
		<post-id xmlns="com-wordpress:feed-additions:1">998</post-id>	</item>
		<item>
		<title>OntoMaton Add-on for Google Sheets</title>
		<link>http://isa-agents.org/2014/11/ontomaton-add-on-for-google-sheets/</link>
		<pubDate>Fri, 14 Nov 2014 10:41:02 +0000</pubDate>
		<dc:creator><![CDATA[agbeltran]]></dc:creator>
				<category><![CDATA[Ontologies]]></category>
		<category><![CDATA[Ontomaton]]></category>
		<category><![CDATA[Uncategorized]]></category>

		<guid isPermaLink="false">http://isaagents.wordpress.com/?p=704</guid>
		<description><![CDATA[This is a very delayed blog post about the OntoMaton Add-on version we released earlier in the year. But better late than never: here we describe the new features we incorporated in the latest OntoMaton version. OntoMaton is a widget bringing together ontology lookup and tagging within the collaborative environment provided by Google Spreadsheets. The [&#8230;]]]></description>
				<content:encoded><![CDATA[<p><a href="https://isa-agents.org/wp-content/uploads/2014/11/ontomaton-fig51.png"><img class="size-medium wp-image-717 aligncenter" src="https://isa-agents.org/wp-content/uploads/2014/11/ontomaton-fig51.png?resize=300%2C62" alt="ontomaton-fig5" width="300" height="62" data-recalc-dims="1" /></a></p>
<p>This is a very delayed blog post about the OntoMaton Add-on version <a href="https://plus.google.com/+EamonnMaguire/posts/LqiezANhRJG">we released earlier in the year</a>. But better late than never: here we describe the new features we incorporated in the latest OntoMaton version.</p>
<p>OntoMaton is a widget bringing together ontology lookup and tagging within the collaborative environment provided by Google Spreadsheets. The original motivation for creating OntoMaton was to support users to create well-annotated experimental metadata in biosciences in a collaborative way, while keeping track of different versions. Google Spreadsheets provide such facilities for collaboration and versioning, so we combined them with ontology search and tagging functionality offered by the <a href="http://www.bioontology.org/wiki/index.php/BioPortal_REST_services">NCBO BioPortal web services</a>. <a href="http://bioportal.bioontology.org/">BioPortal</a> is a web-based repository for biomedical ontologies/terminologies with functionality  for searching and visualizing the ontologies and support ontology-based annotations.</p>
<p>For more information about OntoMaton and its motiviation, see our <a href="http://bioinformatics.oxfordjournals.org/content/29/4/525.long">publication in Bioinformatics</a> and our <a href="http://isaagents.wordpress.com/?s=ontomaton">previous blog posts</a>.</p>
<p>After our initial version of OntoMaton,</p>
<ul>
<li>Google launched <a href="http://googledrive.blogspot.it/2014/03/add-ons.html">Add-ons for Docs and Sheets</a>,  <a href="http://googleappsdeveloper.blogspot.co.uk/2014/06/deprecating-script-gallery-in-old.html">deprecating the Apps Script applications and the Script Gallery</a>, used in the original OntoMaton version</li>
<li>BioPortal released a new API for searching ontology terms and annotator services (see <a href="http://www.bioontology.org/wiki/index.php/BioPortal_Release_Notes#BioPortal_Release_4.0_.28September_2013.29">BioPortal 4.0 release notes</a>), deprecating the old API earlier in 2014</li>
</ul>
<p>Consequently, we upgraded OntoMaton to the latest versions of these services.</p>
<p>We also took the opportunity to incorporate searches across the <a href="http://lov.okfn.org/">Linked Open Vocabularies</a> repository. Linked Open Vocabularies (LOV) is a repository of (<a href="http://www.w3.org/TR/rdf-schema/">RDFS</a> or <a href="http://www.w3.org/TR/owl2-overview/">OWL</a>) vocabularies used in the <a href="http://linkeddata.org/">Linked Data Cloud</a>, and thus, not restricted to bio-ontologies. This addition allows OntoMaton to be used for other use cases, relying on vocabularies outside the bio-domain.</p>
<p>For this new version, the installation procedure with <a href="https://developers.google.com/apps-script/add-ons/">Google Add-ons</a> is as follows:</p>
<ul>
<li>Open a Google Spreadsheet and select the <em>Add-ons</em> menu</li>
</ul>
<p><a href="https://isa-agents.org/wp-content/uploads/2014/11/ontomaton-fig1.png"><img class="size-medium wp-image-709 aligncenter" src="https://isa-agents.org/wp-content/uploads/2014/11/ontomaton-fig1.png?resize=300%2C104" alt="ontomaton-fig1" width="300" height="104" data-recalc-dims="1" /></a></p>
<ul>
<li>Select <em>Get add-ons,</em> search for OntoMaton</li>
</ul>
<p><a href="https://isa-agents.org/wp-content/uploads/2014/11/ontomaton-fig2.png"><img class="size-medium wp-image-710 aligncenter" src="https://isa-agents.org/wp-content/uploads/2014/11/ontomaton-fig2.png?resize=300%2C80" alt="ontomaton-fig2" width="300" height="80" data-recalc-dims="1" /></a></p>
<ul>
<li>By clicking on OntoMaton, you can find more information about it, including some screenshots.</li>
</ul>
<p><a href="https://isa-agents.org/wp-content/uploads/2014/11/ontomaton-fig8.png"><img class="size-medium wp-image-720 aligncenter" src="https://isa-agents.org/wp-content/uploads/2014/11/ontomaton-fig8.png?resize=300%2C239" alt="ontomaton-fig8" width="300" height="239" data-recalc-dims="1" /></a></p>
<ul>
<li>Then you can install OntoMaton (by clicking over the <em>Free</em> button) or if it is already installed, you can manage the installation by clicking on the <em>Manage </em>button.</li>
<li>You will need to authorise OntoMaton to view and manage your spreadsheets (as the Add-on will search over terms from your spreadsheets and incorporate links, etc) and connect to an external service (the REST services that OntoMaton relies upon)</li>
</ul>
<p><a href="https://isa-agents.org/wp-content/uploads/2014/11/ontomaton-fig3.png"><img class="size-medium wp-image-711 aligncenter" src="https://isa-agents.org/wp-content/uploads/2014/11/ontomaton-fig3.png?resize=300%2C291" alt="ontomaton-fig3" width="300" height="291" data-recalc-dims="1" /></a></p>
<ul>
<li> After that, you will be able to use OntoMaton functionality, accessible from the <em>Add-ons</em> menu</li>
</ul>
<p><a href="https://isa-agents.org/wp-content/uploads/2014/11/ontomaton-fig4.png"><img class="size-medium wp-image-712 aligncenter" src="https://isa-agents.org/wp-content/uploads/2014/11/ontomaton-fig4.png?resize=300%2C166" alt="ontomaton-fig4" width="300" height="166" data-recalc-dims="1" /></a></p>
<ul>
<li>And that&#8217;s it! You can start using OntoMaton for searching and tagging&#8230; The functionality is as before, except that when searching you need to select if you want to search BioPortal or LOV.</li>
</ul>
<p><a href="https://isa-agents.org/wp-content/uploads/2014/11/ontomaton-fig61.png"><img class="size-medium wp-image-718 aligncenter" src="https://isa-agents.org/wp-content/uploads/2014/11/ontomaton-fig61.png?resize=300%2C140" alt="ontomaton-fig6" width="300" height="140" data-recalc-dims="1" /></a></p>
<p><a href="https://isa-agents.org/wp-content/uploads/2014/11/ontomaton-fig71.png"><img class="size-medium wp-image-719 aligncenter" src="https://isa-agents.org/wp-content/uploads/2014/11/ontomaton-fig71.png?resize=300%2C164" alt="ontomaton-fig7" width="300" height="164" data-recalc-dims="1" /></a></p>
<p>In the Bioinformatics publication, we shown some of the use cases for OntoMaton. More recently, OntoMaton has been:</p>
<ul>
<li>used to create mappings from the ISA-Tab syntax to several ontologies in our <a href="http://isa-agents.github.io/linkedISA/">linkedISA project</a></li>
<li>used to create nanopublications in <a href="http://isa-agents.github.io/soapdenovo2/">a reproducibility and data reporting case study</a></li>
<li>mentioned in the <a href="https://github.com/jamesaoverton/obo-tutorial/blob/master/docs/using-and-reusing.md#ontomaton">OBO Tutorial for using and re-using ontology terms</a>, by <a href="https://github.com/jamesaoverton">James Overton</a></li>
</ul>
<p>If you are interested in the OntoMaton source code, you can find it in its <a href="https://github.com/ISA-agents/OntoMaton">GitHub repository</a>.</p>
<p>Finally, if you have questions or comments about OntoMaton, contact us (the ISA team) at <em>isaagents &lt;AT&gt; googlegroups.com </em>(replacing &lt;AT&gt; for @!). We would love to hear about how you are using OntoMaton!</p>
]]></content:encoded>
		<post-id xmlns="com-wordpress:feed-additions:1">704</post-id>	</item>
		<item>
		<title>OntoMaton at the International Conference on Biomedical Ontology (ICBO)</title>
		<link>http://isa-agents.org/2013/08/ontomaton-at-the-international-conference-on-biomedical-ontology-icbo/</link>
		<pubDate>Sat, 03 Aug 2013 09:26:20 +0000</pubDate>
		<dc:creator><![CDATA[agbeltran]]></dc:creator>
				<category><![CDATA[Uncategorized]]></category>

		<guid isPermaLink="false">http://isaagents.wordpress.com/?p=570</guid>
		<description><![CDATA[The 4th International Conference on Biomedical Ontology (ICBO 2013) took place in Montreal, Canada, on 8th and 9th July 2013. It was held jointly with the Canadian Semantic Web Symposium (CSWS 2013) and Data Integration in the Life Sciences (DILS 2013) in what was called the Semantic Trilogy 2013. The members of the core ISA team could not [&#8230;]]]></description>
				<content:encoded><![CDATA[<p>The 4th International Conference on Biomedical Ontology (<a href="http://www2.unb.ca/csas/data/ws/icbo2013/">ICBO 2013</a>) took place in Montreal, Canada, on 8th and 9th July 2013. It was held jointly with the Canadian Semantic Web Symposium (CSWS 2013) and Data Integration in the Life Sciences (DILS 2013) in what was called the <a href="http://www2.unb.ca/csas/data/ws/semantic-trilogy-2013/">Semantic Trilogy 2013</a>.</p>
<p>The members of the <a href="http://isatab.sourceforge.net/support.html">core ISA team</a> could not be present this time, but our collaborator and co-author Trish Whetzel was there to present <a href="http://www2.unb.ca/csas/data/ws/icbo2013/highlights.html">OntoMaton in the Highlight Track</a> entitled <a href="http://www2.unb.ca/csas/data/ws/icbo2013/highlights.html"><b>OntoMaton: Google spreadsheets meet NCBO BioPortal services</b></a>. We take this opportunity to thank Trish for her presentation, which was very well received. We have had many new users from the Biomedical Ontology community since ICBO.</p>
<p>The presentation was about our publication &#8220;<b>OntoMaton: a Bioportal powered ontology widget for Google Spreadsheets</b>&#8221; available (open access) in Bioinformatics. 2013 February 15; 29(4): 525–527.</p>
<p>OntoMaton is a script for Google Spreadsheets that relies on <a href="http://www.bioontology.org/wiki/index.php/BioPortal_REST_services">NCBO Bioportal Web Services</a> to provide searching and tagging with ontology terms, using those ontologies registered in <a href="bioportal.bioontology.org">Bioportal</a>. These functionality for ontology-based annotation of spreadsheets can be used collaboratively among distributed parties. Also, Google Spreadsheets provide version control.</p>
<p>This is the link to the publication in Bioinformatics: <a href="http://bioinformatics.oxfordjournals.org/content/29/4/525">http://bioinformatics.oxfordjournals.org/content/29/4/525</a> and this is in PubMed: <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3570217/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3570217/</a>.</p>
<p>And the slides that Trish presented are here:<br />
<iframe src='https://www.slideshare.net/slideshow/embed_code/24842168' width='625' height='512' style='border:1px' allowfullscreen webkitallowfullscreen mozallowfullscreen></iframe></p>
<div style="margin-bottom: 5px;"><strong> <a title="Ontomaton icbo2013-alternative order-t_wv3" href="http://www.slideshare.net/proccaserra/ontomaton-icbo2013alternative-ordertwv3" target="_blank">Ontomaton icbo2013-alternative order-t_wv3</a> </strong> from <strong><a href="http://www.slideshare.net/proccaserra" target="_blank">Philippe Rocca-Serra</a></strong></div>
<div style="margin-bottom: 5px;"></div>
<div style="margin-bottom: 5px;">If you are interested in OntoMaton, apart from the publication and slides, you can also see <a href="http://isaagents.wordpress.com/2012/07/13/introducing-ontomaton-ontology-search-tagging-for-google-spreadsheets/">our previous blog post</a>, <a href="http://www.youtube.com/watch?v=Qs0nxGBfQac&amp;feature=c4-overview&amp;list=UUOR7EUwJ1gQpEvnLEPfRQcw">our YouTube video</a>, some <a href="https://drive.google.com/templates?type=spreadsheets&amp;q=ontomaton">templates</a> and the <a href="https://github.com/ISA-agents/OntoMaton">source code on Github</a>.</div>
<div style="margin-bottom: 5px;"></div>
<div style="margin-bottom: 5px;">Thanks again Trish for presenting OntoMaton at ICBO!</div>
<div style="margin-bottom: 5px;"></div>
]]></content:encoded>
		<post-id xmlns="com-wordpress:feed-additions:1">570</post-id>	</item>
		<item>
		<title>ISAcreator available in GenomeSpace</title>
		<link>http://isa-agents.org/2013/06/isacreator-available-in-genomespace/</link>
		<pubDate>Tue, 18 Jun 2013 17:05:40 +0000</pubDate>
		<dc:creator><![CDATA[agbeltran]]></dc:creator>
				<category><![CDATA[Improvements]]></category>
		<category><![CDATA[ISAcreator]]></category>
		<category><![CDATA[Web]]></category>

		<guid isPermaLink="false">http://isaagents.wordpress.com/?p=540</guid>
		<description><![CDATA[ISAcreator has supported access to GenomeSpace since release 1.7.0. and it is now also available through the GenomeSpace online interface. GenomeSpace is a framework supporting cloud-based interoperability of genomics analysis agents. By providing access to multiple agents through their interface, and supporting file transfers in the cloud, GenomeSpace provides a bridge among the agents, allowing users    Some [&#8230;]]]></description>
				<content:encoded><![CDATA[<p>ISAcreator has supported access to <a href="http://www.genomespace.org/">GenomeSpace</a> since release 1.7.0. and it is now also available through the GenomeSpace online interface.</p>
<p>GenomeSpace is a framework supporting cloud-based interoperability of genomics analysis agents. By providing access to multiple agents through their interface, and supporting file transfers in the cloud, GenomeSpace provides a bridge among the agents, allowing users    Some of the agents available through GenomeSpace are: <a href="http://www.cytoscape.org/">Cytoscape</a>, <a href="http://galaxyproject.org/">Galaxy</a>, <a href="http://www.genepattern.org/">GenePattern</a>, <a href="http://genomica.weizmann.ac.il/">Genomica</a>, <a href="http://www.broadinstitute.org/igv/">Integrative Genomics Viewer (IGV)</a>, and the <a href="http://genome.ucsc.edu/index.html">UCSC Genome Table Browser</a>. Find out more about <a href="http://www.genomespace.org/what-is-genomespace">what is GenomeSpace</a> and what <a href="http://www.genomespace.org/what-can-genomespace-do-for-you">GenomeSpace can do for you</a>.</p>
<p>I will now describe the functionality ISAcreator supports for GenomeSpace.</p>
<p>You can launch ISAcreator from your desktop or you can launch it once you are logged in to GenomeSpace (after <a href="http://www.genomespace.org/register">registering to their service</a>). For launching ISAcreator from within GenomeSpace, just hover over the ISAcreator icon and select &#8216;Launch&#8217;:</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2013/06/gs-8.png"><img class="alignnone  wp-image-551" src="https://isa-agents.org/wp-content/uploads/2013/06/gs-8.png?resize=400%2C67" alt="GS-8" width="400" height="67" data-recalc-dims="1" /></a></p>
<p>If you launch ISAcreator from GenomeSpace, you will be prompted to download ISAcreator and will see the following pop-up window (after you accept to download the file):</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2013/06/gs-10.png"><img src="https://isa-agents.org/wp-content/uploads/2013/06/gs-10.png?resize=300%2C148" alt="GS-10" width="300" height="148" data-recalc-dims="1" /></a></p>
<p>When running ISAcreator (either from your desktop or following the GenomeSpace route), you will notice that it now has a third mode of operation (apart from the previously available light and normal modes) that corresponds to GenomeSpace. With this third method of operation, ISAcreator supports opening ISA-TAB files stored on the cloud environment provided by GenomeSpace and also, saving files into GenomeSpace storage facilities.</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2013/06/gs-7.png"><img class="alignnone size-medium wp-image-549" src="https://isa-agents.org/wp-content/uploads/2013/06/gs-7.png?resize=300%2C177" alt="GS-7" width="300" height="177" data-recalc-dims="1" /></a></p>
<p>If you choose the GenomeSpace mode, you will have to enter your GS user credentials in the ISAcreator login page:</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2013/06/gs-3.png"><img class="alignnone size-medium wp-image-545" src="https://isa-agents.org/wp-content/uploads/2013/06/gs-3.png?resize=300%2C210" alt="GS-3" width="300" height="210" data-recalc-dims="1" /></a></p>
<p>Then you will load the configuration files, as usual, and get to the main menu where you can choose to load an existing ISA-TAB file. If it is not the first time you are loading files, you will see the previously loaded files and also have the option to search GenomeSpace for more files:</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2013/06/gs-11.png"><img class="alignnone size-medium wp-image-559" src="https://isa-agents.org/wp-content/uploads/2013/06/gs-11.png?resize=300%2C210" alt="GS-11" width="300" height="210" data-recalc-dims="1" /></a></p>
<p>As an example, you can find the publicly available BII-I-1 ISA-TAB dataset in GenomeSpace under Public/agbeltran/ISA-TAB-datasets, and select it to load:</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2013/06/gs-12.png"><img class="alignnone size-medium wp-image-560" src="https://isa-agents.org/wp-content/uploads/2013/06/gs-12.png?resize=300%2C210" alt="GS-12" width="300" height="210" data-recalc-dims="1" /></a></p>
<p>After loading an ISA-TAB dataset, you can save it to GenomeSpace (even if it is a local dataset that you want to store in GenomeSpace):</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2013/06/gs-13.png"><img class="alignnone size-medium wp-image-561" src="https://isa-agents.org/wp-content/uploads/2013/06/gs-13.png?resize=300%2C209" alt="GS-13" width="300" height="209" data-recalc-dims="1" /></a></p>
<p>GenomeSpace also provides documentation about ISAcreator in this <a href="http://www.genomespace.org/support/agents/isacreator">page</a> and a guide about using ISAcreator in this <a href="http://www.genomespace.org/support/guides/agent-guide/sections/isacreator">other page</a>.</p>
<p>As always, send us comments or questions contacting:</p>
<ul>
<li>the ISA team at <strong>isaagents [at] googlegroups [dot] com, </strong></li>
<li>the ISA user forum at <strong>isaforum [at] googlegroups [dot] com</strong></li>
</ul>
<p>or send us feature requests or bug reports through the issue tracker in Github:</p>
<ul>
<li><strong>https://github.com/ISA-agents/ISAcreator/issues</strong></li>
</ul>
]]></content:encoded>
		<post-id xmlns="com-wordpress:feed-additions:1">540</post-id>	</item>
		<item>
		<title>New paper on the Metabolights database!</title>
		<link>http://isa-agents.org/2012/11/new-paper-on-the-metabolights-database/</link>
		<pubDate>Fri, 23 Nov 2012 17:11:14 +0000</pubDate>
		<dc:creator><![CDATA[agbeltran]]></dc:creator>
				<category><![CDATA[ISAcommons]]></category>
		<category><![CDATA[Publication]]></category>
		<category><![CDATA[Uncategorized]]></category>

		<guid isPermaLink="false">http://isaagents.wordpress.com/?p=433</guid>
		<description><![CDATA[A recent Nucleic Acids Research (NAR) paper features Metaboligths: the first general-purpose, open access repository for metabolomics studies, storing the studies raw experimental data and associated metadata. You can find the paper here. Metabolights is powered by the the open source ISA metadata framework, and thus is part of the ISA commons, and it is [&#8230;]]]></description>
				<content:encoded><![CDATA[<p>A recent <a href="//nar.oxfordjournals.org”">Nucleic Acids Research</a> (NAR) paper features <a href="//www.ebi.ac.uk/metabolights/”">Metaboligths</a>: the first general-purpose, open access repository for metabolomics studies, storing the studies raw experimental data and associated metadata. You can find the paper <a href="//nar.oxfordjournals.org/content/early/2012/10/28/nar.gks1004">here</a>. Metabolights is powered by the the open source ISA metadata framework, and thus is part of the ISA commons, and it is deployed at the <a href="//www.ebi.ac.uk/”">European Bioinformatics Institute</a> (EBI). </p>
<p>Some of the characteristics of the database are: it is cross-species and cross-techniques, it covers metabolite structures and their reference spectra, their biological roles, locations, concentrations, and raw data from metabolic experiments.</p>
<p>The paper describes the database content and technical architecture, the functionalities for searching and browsing the data, how to download data and get programmatic access to the database, and the data submission process, among other things. </p>
]]></content:encoded>
		<post-id xmlns="com-wordpress:feed-additions:1">433</post-id>	</item>
		<item>
		<title>Risa is out! And available in the Bioconductor 2.11 release&#8230;</title>
		<link>http://isa-agents.org/2012/10/risa-is-out-and-available-in-the-bioconductor-2-11-release/</link>
		<pubDate>Fri, 05 Oct 2012 10:18:06 +0000</pubDate>
		<dc:creator><![CDATA[agbeltran]]></dc:creator>
				<category><![CDATA[Uncategorized]]></category>

		<guid isPermaLink="false">http://isaagents.wordpress.com/?p=412</guid>
		<description><![CDATA[The ISA infrastructure now includes an R package, called Risa, to parse ISAtab datasets into R objects. The Risa package is included in BioConductor release 2.11. Risa provides functionality to read ISA-Tab datasets into R objects. These objects can then be used by downstream Bioconductor packages for data analysis and visualization (i.e, xcms). Currently, metadata associated [&#8230;]]]></description>
				<content:encoded><![CDATA[<p><a href="https://isa-agents.org/wp-content/uploads/2012/10/risa-logo.png"><img align="center" class="alignnone size-full wp-image-416" title="risa-logo" src="https://isa-agents.org/wp-content/uploads/2012/10/risa-logo.png?resize=79%2C72" alt="" width="79" height="72" data-recalc-dims="1" /></a></p>
<p>The ISA infrastructure now includes an R package, called <a href="http://bioconductor.org/packages/release/bioc/html/Risa.html"><strong>Risa</strong></a>, to parse ISAtab datasets into <strong><a href="http://www.r-project.org/">R</a></strong> objects. The <strong>Risa</strong> package is included in <a href="http://www.bioconductor.org/packages/2.11/">BioConductor release 2.11</a>.</p>
<p><strong>Risa</strong> provides functionality to read ISA-Tab datasets into <strong>R</strong> objects. These objects can then be used by downstream <strong>Bioconductor</strong> packages for data analysis and visualization (i.e, xcms). Currently, metadata associated to proteomics and metabolomics-based assays (i.e. mass spectrometry) can be processed into <strong>xcmsSet</strong> objects (from the <strong><a href="http://www.bioconductor.org/packages/release/bioc/html/xcms.html">xcms</a> Bioconductor</strong> package). <strong>Risa</strong> also provides functionality to save the ISA-tab dataset, or each of its individual files, and to update assay files after analysis. For an example of using <strong>Risa </strong>for processing and analysis of metabolomics data, see <a href="https://github.com/sneumann/mtbls2">https://github.com/sneumann/mtbls2</a>.</p>
<p>The source code and latest version can be found in the GitHub repository <a href="https://github.com/ISA-agents/Risa">https://github.com/ISA-agents/Risa</a>. Please, submit all &#8216;bugs&#8217; and feature requests through <a href="https://github.com/ISA-agents/Risa/issues">https://github.com/ISA-agents/Risa/issues</a>.</p>
]]></content:encoded>
		<post-id xmlns="com-wordpress:feed-additions:1">412</post-id>	</item>
	</channel>
</rss>
