<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Web &#8211; ISA agents</title>
	<atom:link href="category/web-2/feed/" rel="self" type="application/rss+xml" />
	<link>http://isa-agents.org</link>
	<description>Standardizing metadata for scientific experiments</description>
	<lastBuildDate>Thu, 19 Apr 2018 15:23:29 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.5</generator>
<site xmlns="com-wordpress:feed-additions:1">113852667</site>	<item>
		<title>Towards a new Sequence Logo Visualization</title>
		<link>http://isa-agents.org/2014/06/towards-a-new-sequence-logo-visualization/</link>
		<pubDate>Mon, 09 Jun 2014 10:44:55 +0000</pubDate>
		<dc:creator><![CDATA[isaagents]]></dc:creator>
				<category><![CDATA[Visualization]]></category>
		<category><![CDATA[Web]]></category>

		<guid isPermaLink="false">http://isaagents.wordpress.com/?p=686</guid>
		<description><![CDATA[Diverging somewhat from our normal posts is a post about a new visualization developed in our group. Eamonn is at EuroVis 2014 in Swansea this week where he will be presenting a paper viewable here Redesigning the Sequence Logo with Glyph-based Approaches to Aid Interpretation on the redesign of the infamous Sequence Logo, some screenshots [&#8230;]]]></description>
				<content:encoded><![CDATA[<p>Diverging somewhat from our normal posts is a post about a new visualization developed in our group.</p>
<p>Eamonn is at <a href="http://eurovis.swansea.ac.uk/" target="_blank">EuroVis 2014</a> in Swansea this week where he will be presenting a paper viewable here <a href="wp-content/uploads/2014/07/sequencelogoredesign.pdf">Redesigning the Sequence Logo with Glyph-based Approaches to Aid Interpretation</a> on the redesign of the infamous Sequence Logo, some screenshots which you can see below. It supports:</p>
<ol>
<li> multiple sequence groups for comparison;</li>
<li>entropy or frequency based encoding of bar size;</li>
<li>glyphs to show changes in position preference for hydropathy or charge;</li>
<li>GestaltLine glyph to show overview of variation at a position; and</li>
<li>Customisation to:
<ol>
<li>reduce transparency of positions where variability is high; or</li>
<li>show the consensus sequence.</li>
</ol>
</li>
</ol>
<div id="attachment_684" style="width: 642px" class="wp-caption aligncenter"><a href="https://isa-agents.org/wp-content/uploads/2014/05/screen-shot-2014-05-14-at-11-49-17.png"><img class="wp-image-682 size-large" src="https://isa-agents.org/wp-content/uploads/2014/05/screen-shot-2014-05-14-at-11-48-11.png?resize=625%2C412" alt="Screen Shot 2014-05-14 at 11.48.11" width="625" height="412" data-recalc-dims="1" /></a><p class="wp-caption-text">Sequence logo with entropy encoding.</p></div>
<p>&nbsp;</p>
<div id="attachment_684" style="width: 642px" class="wp-caption aligncenter"><img class="wp-image-684 size-large" src="https://isa-agents.org/wp-content/uploads/2014/05/screen-shot-2014-05-14-at-11-49-17.png?resize=625%2C412" alt="Screen Shot 2014-05-14 at 11.49.17" width="625" height="412" data-recalc-dims="1" /><p class="wp-caption-text">Sequence logo with frequency based encoding and showing detail on demand view on position hover.</p></div>
<p>&nbsp;</p>
<p>Everything, as always, is open source and available with examples on our GitHub repository</p>
<p><a href="https://github.com/ISA-agents/SequenceLogoVis">https://github.com/ISA-agents/SequenceLogoVis</a></p>
<p>. Let us know what you think, what should be improved, etc. via our issue tracker. The <a href="https://www.dropbox.com/s/04koboj09e381mq/SequenceLogoRedesign-EuroVisSP.pdf" target="_blank">paper is available here</a> and the presentation will be online soon!</p>
]]></content:encoded>
		<post-id xmlns="com-wordpress:feed-additions:1">686</post-id>	</item>
		<item>
		<title>ISAcreator available in GenomeSpace</title>
		<link>http://isa-agents.org/2013/06/isacreator-available-in-genomespace/</link>
		<pubDate>Tue, 18 Jun 2013 17:05:40 +0000</pubDate>
		<dc:creator><![CDATA[agbeltran]]></dc:creator>
				<category><![CDATA[Improvements]]></category>
		<category><![CDATA[ISAcreator]]></category>
		<category><![CDATA[Web]]></category>

		<guid isPermaLink="false">http://isaagents.wordpress.com/?p=540</guid>
		<description><![CDATA[ISAcreator has supported access to GenomeSpace since release 1.7.0. and it is now also available through the GenomeSpace online interface. GenomeSpace is a framework supporting cloud-based interoperability of genomics analysis agents. By providing access to multiple agents through their interface, and supporting file transfers in the cloud, GenomeSpace provides a bridge among the agents, allowing users    Some [&#8230;]]]></description>
				<content:encoded><![CDATA[<p>ISAcreator has supported access to <a href="http://www.genomespace.org/">GenomeSpace</a> since release 1.7.0. and it is now also available through the GenomeSpace online interface.</p>
<p>GenomeSpace is a framework supporting cloud-based interoperability of genomics analysis agents. By providing access to multiple agents through their interface, and supporting file transfers in the cloud, GenomeSpace provides a bridge among the agents, allowing users    Some of the agents available through GenomeSpace are: <a href="http://www.cytoscape.org/">Cytoscape</a>, <a href="http://galaxyproject.org/">Galaxy</a>, <a href="http://www.genepattern.org/">GenePattern</a>, <a href="http://genomica.weizmann.ac.il/">Genomica</a>, <a href="http://www.broadinstitute.org/igv/">Integrative Genomics Viewer (IGV)</a>, and the <a href="http://genome.ucsc.edu/index.html">UCSC Genome Table Browser</a>. Find out more about <a href="http://www.genomespace.org/what-is-genomespace">what is GenomeSpace</a> and what <a href="http://www.genomespace.org/what-can-genomespace-do-for-you">GenomeSpace can do for you</a>.</p>
<p>I will now describe the functionality ISAcreator supports for GenomeSpace.</p>
<p>You can launch ISAcreator from your desktop or you can launch it once you are logged in to GenomeSpace (after <a href="http://www.genomespace.org/register">registering to their service</a>). For launching ISAcreator from within GenomeSpace, just hover over the ISAcreator icon and select &#8216;Launch&#8217;:</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2013/06/gs-8.png"><img class="alignnone  wp-image-551" src="https://isa-agents.org/wp-content/uploads/2013/06/gs-8.png?resize=400%2C67" alt="GS-8" width="400" height="67" data-recalc-dims="1" /></a></p>
<p>If you launch ISAcreator from GenomeSpace, you will be prompted to download ISAcreator and will see the following pop-up window (after you accept to download the file):</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2013/06/gs-10.png"><img src="https://isa-agents.org/wp-content/uploads/2013/06/gs-10.png?resize=300%2C148" alt="GS-10" width="300" height="148" data-recalc-dims="1" /></a></p>
<p>When running ISAcreator (either from your desktop or following the GenomeSpace route), you will notice that it now has a third mode of operation (apart from the previously available light and normal modes) that corresponds to GenomeSpace. With this third method of operation, ISAcreator supports opening ISA-TAB files stored on the cloud environment provided by GenomeSpace and also, saving files into GenomeSpace storage facilities.</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2013/06/gs-7.png"><img class="alignnone size-medium wp-image-549" src="https://isa-agents.org/wp-content/uploads/2013/06/gs-7.png?resize=300%2C177" alt="GS-7" width="300" height="177" data-recalc-dims="1" /></a></p>
<p>If you choose the GenomeSpace mode, you will have to enter your GS user credentials in the ISAcreator login page:</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2013/06/gs-3.png"><img class="alignnone size-medium wp-image-545" src="https://isa-agents.org/wp-content/uploads/2013/06/gs-3.png?resize=300%2C210" alt="GS-3" width="300" height="210" data-recalc-dims="1" /></a></p>
<p>Then you will load the configuration files, as usual, and get to the main menu where you can choose to load an existing ISA-TAB file. If it is not the first time you are loading files, you will see the previously loaded files and also have the option to search GenomeSpace for more files:</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2013/06/gs-11.png"><img class="alignnone size-medium wp-image-559" src="https://isa-agents.org/wp-content/uploads/2013/06/gs-11.png?resize=300%2C210" alt="GS-11" width="300" height="210" data-recalc-dims="1" /></a></p>
<p>As an example, you can find the publicly available BII-I-1 ISA-TAB dataset in GenomeSpace under Public/agbeltran/ISA-TAB-datasets, and select it to load:</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2013/06/gs-12.png"><img class="alignnone size-medium wp-image-560" src="https://isa-agents.org/wp-content/uploads/2013/06/gs-12.png?resize=300%2C210" alt="GS-12" width="300" height="210" data-recalc-dims="1" /></a></p>
<p>After loading an ISA-TAB dataset, you can save it to GenomeSpace (even if it is a local dataset that you want to store in GenomeSpace):</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2013/06/gs-13.png"><img class="alignnone size-medium wp-image-561" src="https://isa-agents.org/wp-content/uploads/2013/06/gs-13.png?resize=300%2C209" alt="GS-13" width="300" height="209" data-recalc-dims="1" /></a></p>
<p>GenomeSpace also provides documentation about ISAcreator in this <a href="http://www.genomespace.org/support/agents/isacreator">page</a> and a guide about using ISAcreator in this <a href="http://www.genomespace.org/support/guides/agent-guide/sections/isacreator">other page</a>.</p>
<p>As always, send us comments or questions contacting:</p>
<ul>
<li>the ISA team at <strong>isaagents [at] googlegroups [dot] com, </strong></li>
<li>the ISA user forum at <strong>isaforum [at] googlegroups [dot] com</strong></li>
</ul>
<p>or send us feature requests or bug reports through the issue tracker in Github:</p>
<ul>
<li><strong>https://github.com/ISA-agents/ISAcreator/issues</strong></li>
</ul>
]]></content:encoded>
		<post-id xmlns="com-wordpress:feed-additions:1">540</post-id>	</item>
		<item>
		<title>Introducing OntoMaton  &#8211; Ontology Search &#038; Tagging for Google Spreadsheets</title>
		<link>http://isa-agents.org/2012/07/introducing-ontomaton-ontology-search-tagging-for-google-spreadsheets/</link>
		<comments>http://isa-agents.org/2012/07/introducing-ontomaton-ontology-search-tagging-for-google-spreadsheets/#comments</comments>
		<pubDate>Fri, 13 Jul 2012 17:50:53 +0000</pubDate>
		<dc:creator><![CDATA[isaagents]]></dc:creator>
				<category><![CDATA[ISA]]></category>
		<category><![CDATA[ISAcreator]]></category>
		<category><![CDATA[Ontologies]]></category>
		<category><![CDATA[Standards]]></category>
		<category><![CDATA[Web]]></category>
		<category><![CDATA[Google docs]]></category>
		<category><![CDATA[Google Spreadsheets]]></category>
		<category><![CDATA[ISA-Ta]]></category>
		<category><![CDATA[ISAagents]]></category>
		<category><![CDATA[NCBO Bioportal]]></category>
		<category><![CDATA[OntoMaton]]></category>

		<guid isPermaLink="false">http://isaagents.wordpress.com/?p=366</guid>
		<description><![CDATA[We are happy to announce the release of OntoMaton, a agent which allows users to search for ontology terms and tag free text right in Google Spreadsheets. This post will serve to introduce you to the agent, how it works and how it can make it easier for users to use ontologies in a pervasive, [&#8230;]]]></description>
				<content:encoded><![CDATA[<p><a href="https://isa-agents.org/wp-content/uploads/2012/07/ontomaton.png"><img class="aligncenter size-medium wp-image-367" title="ontomaton" src="https://isa-agents.org/wp-content/uploads/2012/07/ontomaton.png?resize=300%2C80" alt="" width="300" height="80" data-recalc-dims="1" /></a></p>
<p>We are happy to announce the release of OntoMaton, a agent which allows users to <strong>search for ontology terms</strong> and <strong>tag free text right in Google Spreadsheets</strong>. This post will serve to introduce you to the agent, how it works and how it can make it easier for users to use ontologies in a pervasive, powerful and collaborative environment, complementing existing work from our team in the creation of ISAcreator.</p>
<p><strong>How it looks</strong></p>
<p>OntoMaton is available from the Google Script Gallery and when installed provides a menu as shown below.</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2012/07/menu.png"><img class="aligncenter size-full wp-image-364" title="menu" src="https://isa-agents.org/wp-content/uploads/2012/07/menu.png?resize=625%2C275" alt="" width="625" height="275" data-recalc-dims="1" /></a></p>
<p>From the menu you may access two resources part of OntoMaton: ontology search and ontology tagging. There is also an &#8216;about&#8217; option.</p>
<p><strong>Ontology Search</strong></p>
<p><a href="https://isa-agents.org/wp-content/uploads/2012/07/ontomaton-search-shot.png"><img class="aligncenter size-full wp-image-361" title="ontomaton-search-shot" src="https://isa-agents.org/wp-content/uploads/2012/07/ontomaton-search-shot.png?resize=625%2C681" alt="" width="625" height="681" data-recalc-dims="1" /></a></p>
<p><strong>Ontology Tagging</strong></p>
<p><a href="https://isa-agents.org/wp-content/uploads/2012/07/ontomaton-tag-shot.png"><img class="aligncenter size-full wp-image-360" title="ontomaton-tag-shot" src="https://isa-agents.org/wp-content/uploads/2012/07/ontomaton-tag-shot.png?resize=625%2C476" alt="" width="625" height="476" data-recalc-dims="1" /></a></p>
<p><strong>Behind the scenes: <em>restricting the ontology search space</em></strong></p>
<p>If a sheet named &#8220;restrictions&#8221; is in your spreadsheet, OntoMaton will consult it to determine if the currently selected column/row name has a narrowed ontology search space. This makes it quicker to search BioPortal, allows for restriction of the user&#8217;s result space to make easier the process of selecting a term.</p>
<p><strong>Behind the scenes: <em>extra information about the terms you select</em></strong></p>
<p>For every term you select, it&#8217;s full details are recorded in a &#8220;terms&#8221; sheet. This makes it possible to use OntoMaton in any spreadsheet and all provenance information (including URIs, ontology source and version) for selected ontology terms will be immediately available for use when exposing your records to the <strong>linked data world!</strong></p>
<p><strong>Installing</strong></p>
<p>To install, create a new google spreadsheet, then go to the menu <strong>agents</strong> &gt; <strong>script gallery</strong>. In the script gallery, search for <em>ontology</em> or <em>ontomaton </em>and you&#8217;ll get the following result pane.</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2012/07/scriptgallery.png"><img class="aligncenter size-full wp-image-372" title="scriptgallery" src="https://isa-agents.org/wp-content/uploads/2012/07/scriptgallery.png?resize=625%2C480" alt="" width="625" height="480" data-recalc-dims="1" /></a></p>
<p>Click on &#8216;install&#8217; and this will install the scripts inside your spreadsheet. Then there is one more and final step to follow for installation. You have to click again on <strong>agents</strong> &gt; <strong>script manager </strong>and you&#8217;ll be presented with something like that shown in the image below.</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2012/07/scriptmanager.png"><img class="aligncenter size-full wp-image-371" title="scriptmanager" src="https://isa-agents.org/wp-content/uploads/2012/07/scriptmanager.png?resize=625%2C451" alt="" width="625" height="451" data-recalc-dims="1" /></a></p>
<p>OntoMaton contains lots of functions, but the only one you need to worry about in order to run the program is the <strong><em>onOpen</em></strong> function. Click this then click on <em>run</em> and the OntoMaton menu will be installed in your menu bar. From here you&#8217;ll be able to access the <strong>ontology search</strong> and <strong>ontology tagging</strong> functions.</p>
<p>Let us know what you think! New releases will come soon to fix any problems you may identify, please submit all &#8216;bugs&#8217; and feature requests through <a href="https://github.com/ISA-agents/OntoMaton/issues"><strong>https://github.com/ISA-agents/OntoMaton/issues</strong></a></p>
<p>OntoMaton inherently supports ISA-Tab files too. So if you have an investigation file it will automatically add ontology sources to the ONTOLOGY SOURCE REFERENCE block. Also, if you have Term Source Ref and Term Source Accession after a column, OntoMaton will automatically populate these columns for you.</p>
<p>Also, the following table provides a quick review of available agents attempting to mix spreadsheets and access to vocabulary servers:</p>
<table>
<tbody>
<tr>
<td></td>
<td>
<p dir="ltr">domain</p>
</td>
<td>
<p dir="ltr">automated</p>
<p dir="ltr">annotation</p>
</td>
<td>
<p dir="ltr">ontology search/lookup</p>
</td>
<td>
<p dir="ltr">versioning*</p>
</td>
<td>
<p dir="ltr">collaboration</p>
</td>
</tr>
<tr>
<td>
<p dir="ltr">RightField</p>
</td>
<td>
<p dir="ltr">general</p>
</td>
<td>
<p dir="ltr"><span style="color:red;">✘</span></p>
</td>
<td>
<p dir="ltr"><span style="color:green;">✓</span></p>
</td>
<td>
<p dir="ltr"><span style="color:red;">✘</span></p>
</td>
<td>
<p dir="ltr"><span style="color:red;">✘</span></p>
</td>
</tr>
<tr>
<td>
<p dir="ltr">ISA creator</p>
</td>
<td>
<p dir="ltr">multiomics</p>
</td>
<td>
<p dir="ltr"><span style="color:green;">✓</span></p>
</td>
<td>
<p dir="ltr"><span style="color:green;">✓</span></p>
</td>
<td>
<p dir="ltr"><span style="color:red;">✘</span></p>
</td>
<td>
<p dir="ltr"><span style="color:red;">✘</span></p>
</td>
</tr>
<tr>
<td>
<p dir="ltr">Proteome Harvest PRIDE</p>
</td>
<td>
<p dir="ltr">proteomics</p>
</td>
<td>
<p dir="ltr"><span style="color:red;">✘</span></p>
</td>
<td>
<p dir="ltr"><span style="color:green;">✓</span></p>
</td>
<td>
<p dir="ltr"><span style="color:red;">✘</span></p>
</td>
<td>
<p dir="ltr"><span style="color:red;">✘</span></p>
</td>
</tr>
<tr>
<td>
<p dir="ltr">Annotare</p>
</td>
<td>
<p dir="ltr">transcriptomics</p>
</td>
<td>
<p dir="ltr"><span style="color:red;">✘</span></p>
</td>
<td>
<p dir="ltr"><span style="color:red;">✘</span></p>
</td>
<td>
<p dir="ltr"><span style="color:green;">✓</span></p>
</td>
<td>
<p dir="ltr"><span style="color:red;">✘</span></p>
</td>
</tr>
<tr>
<td>
<p dir="ltr">OntoMaton</p>
</td>
<td>
<p dir="ltr">general</p>
</td>
<td>
<p dir="ltr"><span style="color:green;">✓</span></p>
</td>
<td>
<p dir="ltr"><span style="color:green;">✓</span></p>
</td>
<td>
<p dir="ltr"><span style="color:green;">✓</span></p>
</td>
<td>
<p dir="ltr"><span style="color:green;">✓</span></p>
</td>
</tr>
</tbody>
</table>
<ul>
<li>by versioning we refer to managing of user edits throughout the annotation process.</li>
</ul>
<p>We hope you enjoy this new feature!</p>
<p>The ISA team</p>
<p><i>Addendum:</i></p>
<p><span style="color:red;">Safari 6 users</span>, be aware you will have to activate the &#8216;developer menu&#8217; from the Advanced Item in the Safari &#8216;Preferences&#8217; menu item. Once activated, go to menu &#8216;Develop&#8217; and navigate to &#8216;User Agent&#8217; item and select &#8216;Safari 5.1.7&#8217; for enabling the browser to work with Google Spreadsheet. (Thanks to <a href="https://github.com/rpyzh">rpyzh</a> for reporting the issue, see <a href="https://github.com/ISA-agents/OntoMaton/issues/1">here</a>)</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2012/08/ontomaton-safari-6-preference.png"><img class="aligncenter size-full wp-image-372" title="Ontomaton-Safari-6-preference" src="https://isa-agents.org/wp-content/uploads/2012/08/ontomaton-safari-6-preference.png?resize=625%2C480" alt="" width="625" height="480" data-recalc-dims="1" /></a></p>
<p><a href="https://isa-agents.org/wp-content/uploads/2012/08/ontomaton-safari-6-developer-menu-display.png"><img class="aligncenter" title="Ontomaton-Safari-6-developer-menu-display" src="https://isa-agents.org/wp-content/uploads/2012/08/ontomaton-safari-6-developer-menu-display.png?resize=450%2C300" alt="" width="450" height="300" data-recalc-dims="1" /></a></p>
<p><a href="https://isa-agents.org/wp-content/uploads/2012/08/ontomaton-safari-6-user-agent-selection.png"><img class="aligncenter size-full wp-image-372" title="ontomaton-safari-6-user-agent-selection" src="https://isa-agents.org/wp-content/uploads/2012/08/ontomaton-safari-6-user-agent-selection.png?resize=63%2C48" alt="" width="63" height="48" data-recalc-dims="1" /></a></p>
]]></content:encoded>
			<wfw:commentRss>http://isa-agents.org/2012/07/introducing-ontomaton-ontology-search-tagging-for-google-spreadsheets/feed/</wfw:commentRss>
		<slash:comments>14</slash:comments>
	<post-id xmlns="com-wordpress:feed-additions:1">366</post-id>	</item>
		<item>
		<title>Compared to what? The ArrayExpress Atlas.</title>
		<link>http://isa-agents.org/2011/09/compared-to-what-the-arrayexpress-atlas/</link>
		<comments>http://isa-agents.org/2011/09/compared-to-what-the-arrayexpress-atlas/#comments</comments>
		<pubDate>Mon, 05 Sep 2011 10:23:43 +0000</pubDate>
		<dc:creator><![CDATA[isaagents]]></dc:creator>
				<category><![CDATA[General]]></category>
		<category><![CDATA[Ontologies]]></category>
		<category><![CDATA[Web]]></category>
		<category><![CDATA[ArrayExpress]]></category>
		<category><![CDATA[Atlas]]></category>

		<guid isPermaLink="false">http://isaagents.wordpress.com/?p=250</guid>
		<description><![CDATA[This is intended to be a constructive criticism of a resource which I believe to have the potential to be powerful and useful. Any of you who have read Edward Tufte&#8217;s essay on Visual and Statistical Thinking: Displays of Evidence for Making Decisions will instantly recognise this question&#8230;compared to what? We see many examples in [&#8230;]]]></description>
				<content:encoded><![CDATA[<p>This is intended to be a constructive criticism of a resource which I believe to have the potential to be powerful and useful.</p>
<p>Any of you who have read <strong>Edward Tufte&#8217;s essay on Visual and Statistical Thinking: Displays of Evidence for Making Decisions</strong> will instantly recognise this question&#8230;compared to what? We see many examples in the biological world, and I&#8217;ll focus specifically on one resource here&#8230;the <strong><a href="http://www.ebi.ac.uk/gxa/">ArrayExpress Atlas</a></strong>. <em>First, a disclaimer: I used to work in the group who developed this resource, and have aired my criticisms many years ago to no avail. And not only me, senior researchers have raised the same questions even before the resource was developed, but all suggestions have up to now been ignored.</em></p>
<p>Here, I will only give food for thought about what is presented in the Atlas since some people don&#8217;t seem to understand that what is presented doesn&#8217;t actually make much sense. This is mostly caused by a failure to answer the<strong> compared to what</strong> question&#8230;a particularly important question for a resource which is comparing gene expression levels would you not say?</p>
<p><em><strong>Some examples:</strong></em></p>
<p><strong>The heatmap</strong><br />
A query on the resource, such as <a href="http://www.ebi.ac.uk/gxa/qrs?fact_0=organism_part&amp;fexp_0=UP_DOWN&amp;fmex_0=1&amp;fval_0=liver+&amp;view=hm">this</a> will yield a result like so:</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2011/09/screen-shot-2011-09-05-at-10-34-04.png"><img class="aligncenter size-full wp-image-251" title="Screen shot 2011-09-05 at 10.34.04" src="https://isa-agents.org/wp-content/uploads/2011/09/screen-shot-2011-09-05-at-10-34-04.png?resize=422%2C513" alt="" width="422" height="513" data-recalc-dims="1" /></a></p>
<p>My first thought would be that this heat map is telling me that <strong>Fah</strong> was<strong> up regulated in liver 31 times</strong> and once in some obscure string seemingly encompassing every organism in the human body (I&#8217;ll get to my criticism about these factor representations later). Now, the second question that any self-respecting investigator would ask is compared to what? Is this saying that it is up regulated compared to normal tissue, diseased tissue or all tissue across all organisms? Actually, we don&#8217;t know. And there is nothing to say what is being shown here. Moreover, what does it mean to say up and down regulated. Surely it depends. You can&#8217;t just present discrete variables, one needs to show the statistical meaning of such suggestions&#8230;i.e. show the P value of up/down regulations since not all may be meaningful to a biologist/statistician even though they may well be to guys in the ArrayExpress Atlas team.</p>
<p>Another small point on this is that if this value is dependent on database contents rather than baseline expression levels (whatever they are supposed to be), then if my database contains more liver samples than anything else, and expression levels are calculated relative to this content, my results will be skewed. Either a disclaimer should be presented on the site, or they should make the comparison metrics used more obvious.</p>
<p><strong>The expression profiles &amp; factor display</strong></p>
<p>Based on this <a href="http://www.ebi.ac.uk/gxa/gene/ENSMUSG00000030630">page</a>.</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2011/09/screen-shot-2011-09-05-at-10-51-03.png"><img class="aligncenter size-full wp-image-253" title="Screen shot 2011-09-05 at 10.51.03" src="https://isa-agents.org/wp-content/uploads/2011/09/screen-shot-2011-09-05-at-10-51-03.png?resize=600%2C337" alt="" width="600" height="337" data-recalc-dims="1" /></a>Look at this graph, and tell me what the Y-Axis represents. First of all, even if what they are trying to represent was meaningful, it would still be pretty useless. Let me explain. They have split up variables which are supposed to be related into 3 different tabs, with variables which make NO sense. What does it mean to show time as a variable. Time of what? Sampling time, the length of time an organism was exposed to a compound&#8230;what? Exactly, nothing. It means nothing to show time like this. What does it mean to show dose as a seemingly independent variable. Dosage is no good without a compound. What does make sense and can at least possibly allow one to ask the question<strong> &#8220;compared to what?&#8221;</strong>  is to show <strong>growth factor beta 1 and 5 ng/ml after 1 hour</strong> as one factor, and show the expression levels then (even though we still don&#8217;t know what the Y axis means). You can look at any experiment in the Atlas and find the same problems.</p>
<p><strong>The cluster effect</strong></p>
<p>All people, even those not in the realm of statistics need to understand the importance of the cluster effect. I.e. do I only get over expression of one or more genes when another gene is expressed/under expressed. Transcription networks are indeed networks. There are feedback loops, both positive and negative, and a lot is known about these loops already. So, why are these not taken into account when calculating statistics in the Atlas? For such cases, presenting mutually exclusive P-values of individual genes is not really enough and the clustering effects should be taken into account more so as to adjust the P-value to more realistic sizes.</p>
<p><strong>Summary</strong></p>
<p>I have presented my thoughts on the ArrayExpress Atlas publicly and internally beforehand, but this is the first time I&#8217;m airing it to the public domain. I hope now that something is done to fix this resource since I still believe it to have the potential to be cool and really helpful.</p>
]]></content:encoded>
			<wfw:commentRss>http://isa-agents.org/2011/09/compared-to-what-the-arrayexpress-atlas/feed/</wfw:commentRss>
		<slash:comments>2</slash:comments>
	<post-id xmlns="com-wordpress:feed-additions:1">250</post-id>	</item>
	</channel>
</rss>
