<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>General &#8211; ISA agents</title>
	<atom:link href="category/general/feed/" rel="self" type="application/rss+xml" />
	<link>http://isa-agents.org</link>
	<description>Standardizing metadata for scientific experiments</description>
	<lastBuildDate>Thu, 19 Apr 2018 15:23:29 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.5</generator>
<site xmlns="com-wordpress:feed-additions:1">113852667</site>	<item>
		<title>ISA agenting developed for the metabolomics community</title>
		<link>http://isa-agents.org/2018/03/isa-galaxy-developed-for-metabolomics/</link>
		<pubDate>Fri, 02 Mar 2018 10:44:03 +0000</pubDate>
		<dc:creator><![CDATA[David Johnson]]></dc:creator>
				<category><![CDATA[General]]></category>
		<category><![CDATA[ISA]]></category>
		<category><![CDATA[ISA API]]></category>
		<category><![CDATA[cloud]]></category>
		<category><![CDATA[Galaxy]]></category>
		<category><![CDATA[ISA agents]]></category>
		<category><![CDATA[ISA-Tab]]></category>
		<category><![CDATA[metabolomics]]></category>
		<category><![CDATA[PhenoMeNal]]></category>

		<guid isPermaLink="false">http://isa-agents.org/?p=1316</guid>
		<description><![CDATA[A new set of ISA software agents have been developed out of the EU H2020 PhenoMeNal: Large-Scale Computing for Medical Metabolomics project, which we introduced in this earlier blog post. The 2018-02 release of PhenoMeNal, also known as “Cerebellin”, was released end of February 2018. It represents a major upgrade to the 2017-08 production release. It has a [&#8230;]]]></description>
				<content:encoded><![CDATA[<p>A new set of ISA software agents have been developed out of the EU H2020 <a href="http://phenomenal-h2020.eu">PhenoMeNal: Large-Scale Computing for Medical Metabolomics</a> project, which we introduced in this <a href="2016/03/cloudy-with-a-chance-of-mtbls/">earlier blog post</a>.</p>
<p><img data-attachment-id="1326" data-permalink="2018/03/isa-galaxy-developed-for-metabolomics/phenomenal_logo/" data-orig-file="https://isa-agents.org/wp-content/uploads/2018/03/PhenoMeNal_logo.png?fit=1090%2C206" data-orig-size="1090,206" data-comments-opened="0" data-image-meta="{&quot;aperture&quot;:&quot;0&quot;,&quot;credit&quot;:&quot;&quot;,&quot;camera&quot;:&quot;&quot;,&quot;caption&quot;:&quot;&quot;,&quot;created_timestamp&quot;:&quot;0&quot;,&quot;copyright&quot;:&quot;&quot;,&quot;focal_length&quot;:&quot;0&quot;,&quot;iso&quot;:&quot;0&quot;,&quot;shutter_speed&quot;:&quot;0&quot;,&quot;title&quot;:&quot;&quot;,&quot;orientation&quot;:&quot;0&quot;}" data-image-title="PhenoMeNal project logo" data-image-description="" data-medium-file="https://isa-agents.org/wp-content/uploads/2018/03/PhenoMeNal_logo.png?fit=300%2C57" data-large-file="https://isa-agents.org/wp-content/uploads/2018/03/PhenoMeNal_logo.png?fit=625%2C118" class="aligncenter size-large wp-image-1326" src="https://isa-agents.org/wp-content/uploads/2018/03/PhenoMeNal_logo.png?resize=625%2C118" alt="" width="625" height="118" srcset="https://isa-agents.org/wp-content/uploads/2018/03/PhenoMeNal_logo.png?resize=1024%2C194 1024w, https://isa-agents.org/wp-content/uploads/2018/03/PhenoMeNal_logo.png?resize=300%2C57 300w, https://isa-agents.org/wp-content/uploads/2018/03/PhenoMeNal_logo.png?resize=768%2C145 768w, https://isa-agents.org/wp-content/uploads/2018/03/PhenoMeNal_logo.png?resize=624%2C118 624w, https://isa-agents.org/wp-content/uploads/2018/03/PhenoMeNal_logo.png?w=1090 1090w" sizes="(max-width: 625px) 100vw, 625px" data-recalc-dims="1" /></p>
<p>The 2018-02 release of <em>PhenoMeNal</em>, also known as “<a href="https://github.com/phnmnl/phenomenal-h2020/wiki/Release-notes-2018-02">Cerebellin</a>”, was released end of February 2018. It represents a major upgrade to the 2017-08 production release. It has a richer set of agents, depends on improved deployment software, includes improved workflows for Mass Spectrometry (MS) and Nuclear Magnetic Resonance (NMR) data, and strengthens massively the resilience infrastructure deployments under high load. PhenoMeNal comprises of cloud-based portal infrastructure that includes the <a href="https://galaxyproject.org">Galaxy workflow system</a> customised to run on the <a href="https://kubernetes.io">Kubernetes</a> container orchestrator, with Galaxy agents running their processing in <a href="https://www.docker.com">Docker</a> containers in the cloud. PhenoMeNal, and thus our new ISA Galaxy agents, work in Galaxy running various cloud-computing infrastructures including <a href="https://aws.amazon.com">Amazon Web Services</a>, <a href="https://cloud.google.com">Google Cloud Platform</a>, <a href="https://azure.microsoft.com">Microsoft Azure</a>, <a href="https://www.openstack.org">OpenStack</a> and <a href="https://www.linux-kvm.org">KVM</a>.</p>
<p>The ISA team has been contributing to the project since 2015, and has been collaborating on the development of user-facing, cloud-based data management and processing infrastructure in the project. The <em>Cerebellin</em> release of the PhenoMeNal software includes a new set of ISA-related Galaxy workflow agents, as well as <strong>native support for the ISA-Tab format</strong> in Galaxy. The agents work with the <a href="https://www.ebi.ac.uk/metabolights/">MetaboLights</a> database as well as with ISA-Tab studies uploaded directly into the Galaxy platform, and builds on the <a href="https://github.com/ISA-agents/isa-api/">Python ISA-API</a>.</p>
<div id="attachment_1323" style="width: 635px" class="wp-caption aligncenter"><img data-attachment-id="1323" data-permalink="2018/03/isa-galaxy-developed-for-metabolomics/screen-shot-2018-03-02-at-10-30-05/" data-orig-file="https://isa-agents.org/wp-content/uploads/2018/03/Screen-Shot-2018-03-02-at-10.30.05.png?fit=2560%2C1438" data-orig-size="2560,1438" data-comments-opened="0" data-image-meta="{&quot;aperture&quot;:&quot;0&quot;,&quot;credit&quot;:&quot;&quot;,&quot;camera&quot;:&quot;&quot;,&quot;caption&quot;:&quot;&quot;,&quot;created_timestamp&quot;:&quot;0&quot;,&quot;copyright&quot;:&quot;&quot;,&quot;focal_length&quot;:&quot;0&quot;,&quot;iso&quot;:&quot;0&quot;,&quot;shutter_speed&quot;:&quot;0&quot;,&quot;title&quot;:&quot;&quot;,&quot;orientation&quot;:&quot;0&quot;}" data-image-title="ISA Factors Viz agent in Galaxy" data-image-description="" data-medium-file="https://isa-agents.org/wp-content/uploads/2018/03/Screen-Shot-2018-03-02-at-10.30.05.png?fit=300%2C169" data-large-file="https://isa-agents.org/wp-content/uploads/2018/03/Screen-Shot-2018-03-02-at-10.30.05.png?fit=625%2C351" class="wp-image-1323 size-large" src="https://isa-agents.org/wp-content/uploads/2018/03/Screen-Shot-2018-03-02-at-10.30.05.png?resize=625%2C351" alt="The MetaboLights/ISA-Tab Factors Visualization agent in Galaxy." width="625" height="351" srcset="https://isa-agents.org/wp-content/uploads/2018/03/Screen-Shot-2018-03-02-at-10.30.05.png?resize=1024%2C575 1024w, https://isa-agents.org/wp-content/uploads/2018/03/Screen-Shot-2018-03-02-at-10.30.05.png?resize=300%2C169 300w, https://isa-agents.org/wp-content/uploads/2018/03/Screen-Shot-2018-03-02-at-10.30.05.png?resize=768%2C431 768w, https://isa-agents.org/wp-content/uploads/2018/03/Screen-Shot-2018-03-02-at-10.30.05.png?resize=624%2C351 624w, https://isa-agents.org/wp-content/uploads/2018/03/Screen-Shot-2018-03-02-at-10.30.05.png?w=1250 1250w, https://isa-agents.org/wp-content/uploads/2018/03/Screen-Shot-2018-03-02-at-10.30.05.png?w=1875 1875w" sizes="(max-width: 625px) 100vw, 625px" data-recalc-dims="1" /><p class="wp-caption-text">The MetaboLights/ISA-Tab Factors Visualization agent in Galaxy.</p></div>
<p>The new ISA Galaxy agents include:</p>
<ul>
<li>Metabolights downloader (W4M), developed with our colleagues at <a href="http://www.cea.fr">CEA</a>, downloads MetaboLights studies in the new Galaxy &#8220;isa-tab&#8221; data type.</li>
<li>Study Metadata Exploration agents (5 different agents) that allows querying over ISA-Tab data based on study factor slicing.</li>
<li>MetaboLights Factors Viz &#8211; a agent developed with our colleagues at EMBL-EBI for visualizing a summary of study factors as a <a href="https://www.jasondavies.com/parallel-sets/">parallel sets plot</a>.</li>
<li>Format conversions from ISA-Tab (using the &#8220;isa-tab&#8221; Galaxy data type) to ISA-JSON and to W4M (developed by <a href="http://www.cea.fr">CEA</a>).</li>
<li>ISA-Tab validation, again using the &#8220;isa-tab&#8221; Galaxy data type.</li>
<li>mzml2isa and nmrml2isa &#8211; Automated study metadata creation in ISA-Tab using the &#8220;isa-tab&#8221; Galaxy data type, from mlML and nmrML data, developed with our colleagues at the University of Birmingham.</li>
<li>And finally, an interactive agent to create prospective ISA-Tab study templates as &#8220;isa-tab&#8221; Galaxy data types, based on study design information. This agent supports generating assays for both MS and NMR, using standardised file naming templates compatible with <a href="https://www.birmingham.ac.uk/research/activity/phenome-centre/">Phenome Centre Birmingham</a> and the <a href="http://www.imperial.ac.uk/phenome-centre">MRC-NIHR National Phenome Centre</a> at Imperial College London. The agent shares curation practices with those used by the <a href="https://fairsharing.org/FAIRsharing.exz30t">MetaboLights</a> database and implements the <a href="https://fairsharing.org/FAIRsharing.exz30t">Metabolomics Standards Initiative</a> (MSI) reporting guidelines that go towards making metadata and data <a href="https://doi.org/10.1038/sdata.2016.18">FAIR</a>.</li>
</ul>
<p>We are also developing extensions to our Galaxy agents to support NGS and DNA microarray data, and to enable direct deposition to public repositories, such as those hosted by EMBL-EBI, via Galaxy workflows.</p>
<p>You can try out our ISA Galaxy agents in the <em>Cerebellin</em> release of PhenoMeNal in the <a href="https://public.phenomenal-h2020.eu">public PhenoMeNal Galaxy server</a>. The next scheduled release of PhenoMeNal will be the <em>Dalcotidine</em> release scheduled for August 2018.</p>
]]></content:encoded>
		<post-id xmlns="com-wordpress:feed-additions:1">1316</post-id>	</item>
		<item>
		<title>ISAcreator 1.7.11 now available</title>
		<link>http://isa-agents.org/2017/12/isacreator-1-7-11-now-available/</link>
		<comments>http://isa-agents.org/2017/12/isacreator-1-7-11-now-available/#comments</comments>
		<pubDate>Wed, 06 Dec 2017 15:21:58 +0000</pubDate>
		<dc:creator><![CDATA[David Johnson]]></dc:creator>
				<category><![CDATA[General]]></category>
		<category><![CDATA[Improvements]]></category>
		<category><![CDATA[ISA]]></category>
		<category><![CDATA[ISAcreator]]></category>
		<category><![CDATA[Java]]></category>

		<guid isPermaLink="false">http://isa-agents.org/?p=1294</guid>
		<description><![CDATA[Today we announce the release of ISAcreator 1.7.11. This release updates ISAcreator to work with Java 9. You can download ISAcreator 1.7.11 from Github here: https://github.com/ISA-agents/ISAcreator/releases/tag/v1.7.11 If you&#8217;re an ISAcreator user, or use any of the other ISA-agents suite, please let us know and we can list you as being part of the ISAcommons community. If [&#8230;]]]></description>
				<content:encoded><![CDATA[<p>Today we announce the release of <strong><a href="https://github.com/ISA-agents/ISAcreator/releases/tag/v1.7.11">ISAcreator 1.7.11</a></strong>.</p>
<p>This release updates ISAcreator to work with Java 9.</p>
<p><img data-attachment-id="1295" data-permalink="2017/12/isacreator-1-7-11-now-available/isacreator1711/" data-orig-file="https://isa-agents.org/wp-content/uploads/2017/12/ISAcreator1711.png?fit=2272%2C1404" data-orig-size="2272,1404" data-comments-opened="0" data-image-meta="{&quot;aperture&quot;:&quot;0&quot;,&quot;credit&quot;:&quot;&quot;,&quot;camera&quot;:&quot;&quot;,&quot;caption&quot;:&quot;&quot;,&quot;created_timestamp&quot;:&quot;0&quot;,&quot;copyright&quot;:&quot;&quot;,&quot;focal_length&quot;:&quot;0&quot;,&quot;iso&quot;:&quot;0&quot;,&quot;shutter_speed&quot;:&quot;0&quot;,&quot;title&quot;:&quot;&quot;,&quot;orientation&quot;:&quot;0&quot;}" data-image-title="ISAcreator Poznań" data-image-description="" data-medium-file="https://isa-agents.org/wp-content/uploads/2017/12/ISAcreator1711.png?fit=300%2C185" data-large-file="https://isa-agents.org/wp-content/uploads/2017/12/ISAcreator1711.png?fit=625%2C386" class="aligncenter wp-image-1295 size-large" src="https://isa-agents.org/wp-content/uploads/2017/12/ISAcreator1711.png?resize=625%2C386" alt="" width="625" height="386" srcset="https://isa-agents.org/wp-content/uploads/2017/12/ISAcreator1711.png?resize=1024%2C633 1024w, https://isa-agents.org/wp-content/uploads/2017/12/ISAcreator1711.png?resize=300%2C185 300w, https://isa-agents.org/wp-content/uploads/2017/12/ISAcreator1711.png?resize=768%2C475 768w, https://isa-agents.org/wp-content/uploads/2017/12/ISAcreator1711.png?resize=624%2C386 624w, https://isa-agents.org/wp-content/uploads/2017/12/ISAcreator1711.png?w=1250 1250w, https://isa-agents.org/wp-content/uploads/2017/12/ISAcreator1711.png?w=1875 1875w" sizes="(max-width: 625px) 100vw, 625px" data-recalc-dims="1" /></p>
<p>You can download ISAcreator 1.7.11 from Github here: <a href="https://github.com/ISA-agents/ISAcreator/releases/tag/v1.7.11">https://github.com/ISA-agents/ISAcreator/releases/tag/v1.7.11</a></p>
<p>If you&#8217;re an ISAcreator user, or use any of the other ISA-agents suite, please let us know and we can list you as being part of the <a href="http://isacommons.org"><em>ISAcommons</em></a> community.</p>
<p>If you have any questions or any problems with using ISAcreator, please drop the ISA Team an email to <a href="mailto:isaagents@googlegroups.com">isaagents@googlegroups.com</a> or to the <a href="https://groups.google.com/forum/#!forum/isaforum">ISA community forum</a>.</p>
]]></content:encoded>
			<wfw:commentRss>http://isa-agents.org/2017/12/isacreator-1-7-11-now-available/feed/</wfw:commentRss>
		<slash:comments>2</slash:comments>
	<post-id xmlns="com-wordpress:feed-additions:1">1294</post-id>	</item>
		<item>
		<title>Plant Science takes a focus on ISA</title>
		<link>http://isa-agents.org/2016/07/plant-science-takes-a-focus-on-isa/</link>
		<pubDate>Wed, 06 Jul 2016 10:49:36 +0000</pubDate>
		<dc:creator><![CDATA[David Johnson]]></dc:creator>
				<category><![CDATA[General]]></category>
		<category><![CDATA[ISA]]></category>
		<category><![CDATA[Standards]]></category>
		<category><![CDATA[workshops]]></category>
		<category><![CDATA[arabidopsis]]></category>
		<category><![CDATA[BioSharing]]></category>
		<category><![CDATA[FAIRdata]]></category>
		<category><![CDATA[Isa]]></category>
		<category><![CDATA[ISA infrastructure]]></category>
		<category><![CDATA[plant science]]></category>

		<guid isPermaLink="false">http://isa-agents.org/?p=1150</guid>
		<description><![CDATA[Back in April this year, Dr David Johnson from the ISA team gave a presentation on &#8220;Data Infrastructures to Foster Data Reuse&#8221; at a workshop on Integrating Large Data into Plant Science: From Big Data to Discovery hosted by GARnet (the UK network for Arabidopsis researchers) and Egenis (the Exeter Centre for the Study of [&#8230;]]]></description>
				<content:encoded><![CDATA[<p>Back in April this year, Dr David Johnson from the ISA team gave a presentation on &#8220;Data Infrastructures to Foster Data Reuse&#8221; at a workshop on <a href="http://www.plantsci.org.uk/events/integrating-large-data-plant-science-big-data-discovery"><em>Integrating Large Data into Plant Science: From Big Data to Discovery</em></a> hosted by <a href="http://www.garnetcommunity.org.uk/">GARnet</a> (the UK network for Arabidopsis researchers) and <a href="http://socialsciences.exeter.ac.uk/sociology/research/sts/egenis/">Egenis</a> (the Exeter Centre for the Study of the Life Sciences). The workshop was held at Dartington Hall in Devon, South West England, and was well attended by researchers from the plant and biological science community worldwide as well as representatives from industry from organisations such as <a href="https://www.syngenta.co.uk/">Syngenta</a>.</p>
<p>David presented on ISA, as well as on <a href="http://biosharing.org">biosharing.org</a>, as candidate data infrastructure resources for enabling data reuse in the plant sciences, as well as presenting an example of how one might encode high-throughput plant phenotyping in ISA tab.</p>
<p>We have observed the uptake of the ISA tab format across the broad range of life sciences, but view its adoption, with a view to making data <a href="http://www.nature.com/articles/sdata201618">FAIR</a> (Findable, Accessible, Interoperable and Reusable), in the plant sciences as essential for the field. In particular centres such as the UK’s <a href="http://www.plant-phenomics.ac.uk">National Plant Phenomics Centre</a> in Aberystwyth, Wales, could benefit hugely from adopting ISA where there are emerging challenges in data management, in particular as automation of data collection is a significant driver in modern plant-based research and agritech.</p>
<p>There are also existing data analysis platforms such as <a href="https://www.araport.org/">Araport</a> (the Arabidopsis information Portal), <a href="http://www.phoenixbioinformatics.org/tair/">TAIR</a> (The Arabidopsis Information Resources) and <a href="https://www.biodare.ed.ac.uk/">BioDare</a> (Biological Data Repository) that could benefit from standardizing their experimental data, as well as ongoing efforts to create open data resources in the plant sciences, such as the <a href="http://copo-project.org/">Collaborative Open Plant Omics</a> (COPO) project, that will be using the new ISA JSON format as native data objects.</p>
<p>You can check out David’s presentation <a href="http://www.slideshare.net/DavidJohnson478/garnet-workshop-on-integrating-large-data-into-plant-science">on SlideShare</a>.</p>
<p><iframe src="https://www.slideshare.net/slideshow/embed_code/key/EwVUZUiuUcN7HB" width="427" height="356" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; margin-bottom:5px; max-width: 100%;" allowfullscreen> </iframe> </p>
<div style="margin-bottom:5px"> <strong> <a href="https://www.slideshare.net/DavidJohnson478/garnet-workshop-on-integrating-large-data-into-plant-science" title="GARNet workshop on Integrating Large Data into Plant Science" target="_blank">GARNet workshop on Integrating Large Data into Plant Science</a> </strong> from <strong><a target="_blank" href="http://www.slideshare.net/DavidJohnson478">David Johnson</a></strong> </div>
]]></content:encoded>
		<post-id xmlns="com-wordpress:feed-additions:1">1150</post-id>	</item>
		<item>
		<title>Compared to what? The ArrayExpress Atlas.</title>
		<link>http://isa-agents.org/2011/09/compared-to-what-the-arrayexpress-atlas/</link>
		<comments>http://isa-agents.org/2011/09/compared-to-what-the-arrayexpress-atlas/#comments</comments>
		<pubDate>Mon, 05 Sep 2011 10:23:43 +0000</pubDate>
		<dc:creator><![CDATA[isaagents]]></dc:creator>
				<category><![CDATA[General]]></category>
		<category><![CDATA[Ontologies]]></category>
		<category><![CDATA[Web]]></category>
		<category><![CDATA[ArrayExpress]]></category>
		<category><![CDATA[Atlas]]></category>

		<guid isPermaLink="false">http://isaagents.wordpress.com/?p=250</guid>
		<description><![CDATA[This is intended to be a constructive criticism of a resource which I believe to have the potential to be powerful and useful. Any of you who have read Edward Tufte&#8217;s essay on Visual and Statistical Thinking: Displays of Evidence for Making Decisions will instantly recognise this question&#8230;compared to what? We see many examples in [&#8230;]]]></description>
				<content:encoded><![CDATA[<p>This is intended to be a constructive criticism of a resource which I believe to have the potential to be powerful and useful.</p>
<p>Any of you who have read <strong>Edward Tufte&#8217;s essay on Visual and Statistical Thinking: Displays of Evidence for Making Decisions</strong> will instantly recognise this question&#8230;compared to what? We see many examples in the biological world, and I&#8217;ll focus specifically on one resource here&#8230;the <strong><a href="http://www.ebi.ac.uk/gxa/">ArrayExpress Atlas</a></strong>. <em>First, a disclaimer: I used to work in the group who developed this resource, and have aired my criticisms many years ago to no avail. And not only me, senior researchers have raised the same questions even before the resource was developed, but all suggestions have up to now been ignored.</em></p>
<p>Here, I will only give food for thought about what is presented in the Atlas since some people don&#8217;t seem to understand that what is presented doesn&#8217;t actually make much sense. This is mostly caused by a failure to answer the<strong> compared to what</strong> question&#8230;a particularly important question for a resource which is comparing gene expression levels would you not say?</p>
<p><em><strong>Some examples:</strong></em></p>
<p><strong>The heatmap</strong><br />
A query on the resource, such as <a href="http://www.ebi.ac.uk/gxa/qrs?fact_0=organism_part&amp;fexp_0=UP_DOWN&amp;fmex_0=1&amp;fval_0=liver+&amp;view=hm">this</a> will yield a result like so:</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2011/09/screen-shot-2011-09-05-at-10-34-04.png"><img class="aligncenter size-full wp-image-251" title="Screen shot 2011-09-05 at 10.34.04" src="https://isa-agents.org/wp-content/uploads/2011/09/screen-shot-2011-09-05-at-10-34-04.png?resize=422%2C513" alt="" width="422" height="513" data-recalc-dims="1" /></a></p>
<p>My first thought would be that this heat map is telling me that <strong>Fah</strong> was<strong> up regulated in liver 31 times</strong> and once in some obscure string seemingly encompassing every organism in the human body (I&#8217;ll get to my criticism about these factor representations later). Now, the second question that any self-respecting investigator would ask is compared to what? Is this saying that it is up regulated compared to normal tissue, diseased tissue or all tissue across all organisms? Actually, we don&#8217;t know. And there is nothing to say what is being shown here. Moreover, what does it mean to say up and down regulated. Surely it depends. You can&#8217;t just present discrete variables, one needs to show the statistical meaning of such suggestions&#8230;i.e. show the P value of up/down regulations since not all may be meaningful to a biologist/statistician even though they may well be to guys in the ArrayExpress Atlas team.</p>
<p>Another small point on this is that if this value is dependent on database contents rather than baseline expression levels (whatever they are supposed to be), then if my database contains more liver samples than anything else, and expression levels are calculated relative to this content, my results will be skewed. Either a disclaimer should be presented on the site, or they should make the comparison metrics used more obvious.</p>
<p><strong>The expression profiles &amp; factor display</strong></p>
<p>Based on this <a href="http://www.ebi.ac.uk/gxa/gene/ENSMUSG00000030630">page</a>.</p>
<p><a href="https://isa-agents.org/wp-content/uploads/2011/09/screen-shot-2011-09-05-at-10-51-03.png"><img class="aligncenter size-full wp-image-253" title="Screen shot 2011-09-05 at 10.51.03" src="https://isa-agents.org/wp-content/uploads/2011/09/screen-shot-2011-09-05-at-10-51-03.png?resize=600%2C337" alt="" width="600" height="337" data-recalc-dims="1" /></a>Look at this graph, and tell me what the Y-Axis represents. First of all, even if what they are trying to represent was meaningful, it would still be pretty useless. Let me explain. They have split up variables which are supposed to be related into 3 different tabs, with variables which make NO sense. What does it mean to show time as a variable. Time of what? Sampling time, the length of time an organism was exposed to a compound&#8230;what? Exactly, nothing. It means nothing to show time like this. What does it mean to show dose as a seemingly independent variable. Dosage is no good without a compound. What does make sense and can at least possibly allow one to ask the question<strong> &#8220;compared to what?&#8221;</strong>  is to show <strong>growth factor beta 1 and 5 ng/ml after 1 hour</strong> as one factor, and show the expression levels then (even though we still don&#8217;t know what the Y axis means). You can look at any experiment in the Atlas and find the same problems.</p>
<p><strong>The cluster effect</strong></p>
<p>All people, even those not in the realm of statistics need to understand the importance of the cluster effect. I.e. do I only get over expression of one or more genes when another gene is expressed/under expressed. Transcription networks are indeed networks. There are feedback loops, both positive and negative, and a lot is known about these loops already. So, why are these not taken into account when calculating statistics in the Atlas? For such cases, presenting mutually exclusive P-values of individual genes is not really enough and the clustering effects should be taken into account more so as to adjust the P-value to more realistic sizes.</p>
<p><strong>Summary</strong></p>
<p>I have presented my thoughts on the ArrayExpress Atlas publicly and internally beforehand, but this is the first time I&#8217;m airing it to the public domain. I hope now that something is done to fix this resource since I still believe it to have the potential to be cool and really helpful.</p>
]]></content:encoded>
			<wfw:commentRss>http://isa-agents.org/2011/09/compared-to-what-the-arrayexpress-atlas/feed/</wfw:commentRss>
		<slash:comments>2</slash:comments>
	<post-id xmlns="com-wordpress:feed-additions:1">250</post-id>	</item>
	</channel>
</rss>
